<!doctype html><html><head><title>Séminaire virtuel: vendredi 2 septembre 2022</title><meta charset=utf-8><meta name=X-UA-Compatible content="IE=edge"><meta name=google-site-verification content><meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport><meta content="telephone=no" name=format-detection><meta name=description content><meta name=renderer content="webkit"><meta name=theme-color content="#ffffff"><meta property="og:url" content="https://gt-bioss.cnrs.fr/seminaires/2022-09/"><meta property="og:site_name" content="GT Bioss"><meta property="og:title" content="Séminaire virtuel: vendredi 2 septembre 2022"><meta property="og:description" content="Carito Guziolowski (LS2N, École centrale de Nantes)"><meta property="og:locale" content="fr_FR"><meta property="og:type" content="article"><meta property="article:section" content="seminaires"><meta property="article:published_time" content="2022-09-02T13:00:00+00:00"><meta property="article:modified_time" content="2022-09-02T13:00:00+00:00"><link type=text/css rel=stylesheet href=/vendor/css/bootstrap.min.css><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/journal.min.ea4dd62a23031df613ee03df707ef6cbfa53acf19bdc53e6ba916edc2504def4.css integrity="sha256-6k3WKiMDHfYT7gPfcH72y/pTrPGb3FPmupFu3CUE3vQ=" media=screen><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/dark-mode.min.0286a9cefa1dd841baa4957e10f75bb71af3099ccebeadaf249daebf5842ceb4.css integrity="sha256-Aoapzvod2EG6pJV+EPdbtxrzCZzOvq2vJJ2uv1hCzrQ=" media=screen><script src=/vendor/js/loadCSS.js></script><script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script><script src=/js/toc.js></script><script>console.log("Hello from 'layouts/partials/extended_head.html'")</script></head><body><div id=app><div id=sideContainer class=side-container><a class="a-block nav-head false" href=https://gt-bioss.cnrs.fr/><div class=nav-title>GT Bioss</div><div class=nav-subtitle>Groupe de travail sur la biologie systémique symbolique</div></a><div class=nav-link-list><a class="a-block nav-link-item false" href=/about>À propos
</a><a class="a-block nav-link-item active" href=/seminaires/>Séminaires mensuels
</a><a class="a-block nav-link-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block nav-link-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block nav-link-item false" href=/inscription/>Inscription
</a><a class="a-block nav-link-item false" href=/contact.html>Contact</a></div><div class=nav-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><div id=extraContainer class=extra-container><div class=pagination><a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }"><i class="material-icons pagination-action-icon">keyboard_arrow_up
</i></a><a type=button class=pagination-action id=darkModeToggleButton><span class="material-icons pagination-action-icon" id=darkModeToggleIcon>dark_mode</span></a></div></div><div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }"><div class=drawer-content><div class=drawer-menu><a class="a-block drawer-menu-item false" href=/about>À propos
</a><a class="a-block drawer-menu-item active" href=/seminaires/>Séminaires mensuels
</a><a class="a-block drawer-menu-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block drawer-menu-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block drawer-menu-item false" href=/inscription/>Inscription
</a><a class="a-block drawer-menu-item false" href=/contact.html>Contact</a></div></div></div><transition name=fade><div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div></transition><nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container"><div id=navBackground class=nav-background></div><div class="container container-narrow nav-content"><button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>menu
</i></button>
<a id=navTitle class=navbar-brand href=https://gt-bioss.cnrs.fr/>GT Bioss
</a><button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>dark_mode</i></button></div></nav><div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }"><a href=https://gt-bioss.cnrs.fr/><div class=single-column-header-title>GT Bioss</div><div class=single-column-header-subtitle>Groupe de travail sur la biologie systémique symbolique</div></a></div><div id=content><div id=streamContainer class=stream-container><div class="post-list-container post-list-container-shadow"><div class=post><div class=post-head-wrapper-text-only><div class=post-title>Séminaire virtuel: vendredi 2 septembre 2022<div class=post-subtitle>Carito Guziolowski (LS2N, École centrale de Nantes)</div><div class=post-meta></div></div></div><div class=post-body-wrapper><div class=post-body v-pre><p><a href="https://u-bordeaux-fr.zoom.us/j/86764096440?pwd=b01qOG04RTMvRWNOVHBYR1ZIbkVaUT09">Lien Zoom</a></p><ul><li>Meeting ID: 867 6409 6440</li><li>passcode: 149120</li></ul><h3 id=13h00---13h50----carito-guziolowski-ls2n-école-centrale-de-nantes>13h00 - 13h50 &ndash; Carito Guziolowski (LS2N, École centrale de Nantes)</h3><p><strong>Machine Learning, Logic Programming and Learning Boolean Networks</strong></p><p>This talk will present a work published on 2018 (Chebouba et al., BMC Bioinformatics),
applied to discriminate the response of Acute Myeloid Leukemia (AML) patients. The data we
used was obtained from the DREAM 9 challenge, specifically: given a phosphoproteomic
dataset composed of 231 measured proteins across 191 AML individuals, and knowing which
prognostic (“complete remission” or “resistant”) the individuals had after receiving treatment,
find the computational model that fits both prognostics. Our model was a family of Boolean
networks (BNs) that explained the data of remission/resistant classes of AML patients. We
later confirmed that the different rewiring of protein PIK3CA present in the learned BN
families, confirmed biological reality.<br>The core of this research focused on how phosphoproteomic data was handled in order to
produce discriminant data points. Such discriminant data points constitute an essential input
of a tool, caspo, we developed to learn BNs. Retrieving discriminant models is classically done
by machine learning and statistical methods. For the Dream 9 challenge these methods were
proposed and focused mainly on clinical variables, since they found them more discriminant of
AML patient response to treatment than proteomics data. Our graph-based approach applied
a machine learnign logical program to obtain a discriminant dataset, used later by caspo to
discover boolean gates between discriminant species connected in a prior regulatory network.<br>This approach is interesting in our current research because it allows to obtain discriminant
input-data, focusing on a specific subset of species across a specific subset of individuals, to
learn BNs. It relied on the high dimensionality of the dataset: 231 species measured for 191
AML individuals. Similar problems, for example considering single-cell data, have larger
dimensions (e.g. 30000 genes across 1500 cells) and may generate multiple choices for input-
data as well. Current machine learning approaches to handle single-cell data focus on
similarites or co-expression regions (e.g. WGCNA) to reduce the dimension of genes. We
believe it can be also interesting to focus on the differences, mutual-exclusiveness and
contrasts of single-cell expression, across multiple states to select genes behaviors that allow
us to learn a consensual model or families of BNs that explain different paths taken by cells to
arrive to the same cell-fate.</p><hr width=100% id=EOF><p style=color:#777>Dernière modification le 02/09/2022</p></div></div><nav class=post-pagination><a class=newer-posts href=https://gt-bioss.cnrs.fr/seminaires/2022-10/>Suivant<br>Séminaire virtuel: vendredi 7 octobre 2022
</a><a class=older-posts href=https://gt-bioss.cnrs.fr/events/hybrid22/>Précédent<br>Workshop Hybrid models and methods in systems medicine</a></nav><div class=post-comment-wrapper></div></div></div></div></div><div id=single-column-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><script src=/js/journal.js></script></body></html>