---
date: 2023-07-03
title: "Thematic workshop on networks and biological models inference"
summary: "Le **3 et 4 juillet 2023** à **Marseille**"
categories: "Manifestations"
---

The aim is to gather people working on this topic to present recent results and discuss challenges and perspectives. The workshop will take place on 3 and 4 July 2023 at CIRM in Marseille. 

We are grateful to GDR [IM](https://www.gdr-im.fr/), [BIM](https://www.gdr-bim.cnrs.fr/), [RADIA](https://www.gdria.fr/), and RT [MathSAV](https://mathsav.math.cnrs.fr) for their financial support.

### Registration

Free, but mandatory. \
⚠️ The number of places being (very) limited, we will confirm your registration as soon as possible.

https://framaforms.org/bioss-workshop-on-networks-and-biological-models-inference-1682358435

### Scientific Program

This is a tentative program, yet to be confirmed.

**Monday, July 3rd** 

Morning session 10:00-12:00

- 10:00 - 10:30: Malvina Marku (INSERM, CRCT), *Data-driven regulatory network inference, an introduction*
- 10:30 - 11:00: Paola Vera Licona (UConn Health)
- 11:00 - 11:30: Ulysse Herbach (Inria, Nancy), *Gene expression and regulatory networks: bridging the gap between mechanistic modeling and statistical learning*
- 11:30 - 12:00: Maxime Folschette (Centrale Lille Institut), *Learning Biological Regulatory Networks from Time Series with LFIT: Theory and Practice*

Afternoon session 12:00 - 16:00

- 14:00 - 14:30: Mathieu Bolteau (LS2N Univ Nantes)
- 14:30 - 15:00: Loïc Paulevé (CNRS/LaBRI Univ Bordeaux)
- 15:00 - 15:30: Athénaïs Vaginay (Inria Nancy)
- 15:30 - 16:00: Laetitia Gibart (I3S, Univ Côte d'Azur)

Discussion 16:30 - 17:30

**Tuesday, July 4rth** 

Morning session 09:30 - 12:00

- 09:30 - 10:00: Anaïs Baudaot (MMG, Univ Aix Marseille), *Network-based multimodal data integration*
- 10:00 - 10:30: Mathieu Hemery (Inria Saclay), *Artificial CRN synthesis from function specification and natural CRN repository screening*
- 10:30 - 11:00: François Fages (Inria Saclay), *Equation discovery and chemical reaction model learning from data time series*
- 11:00 - 12:00: Wrap-up discussion, and concluding remarks


### Abstracts

By order of appearance

**Malvina Marku (INSERM, CRCT)**\
*Data-driven regulatory network inference, an introduction*\
Dissecting cellular reprogramming and crosstalk through gene regulatory networks
Gene regulatory networks are highly valuable conceptual models of biological systems. The inference of GRNs has posed a significant challenge in the field of systems biology for approximately two decades. In an ideal scenario, a comprehensive collection of GRNs would exist, encompassing all cell types and transcription factors, obtained through chromatin immunoprecipitation (ChIP-seq) followed by sequencing. Establishing GRNs in engineered cells would involve quantifying their overall transcription factor binding profiles and comparing them to the reference dataset. Unfortunately, such data is currently unavailable. However, recently published methods for reconstructing GRNs using genome-wide expression data have greatly improved. Additionally, repositories of gene expression have accumulated diverse biological perturbations, which are essential for accurately reconstructing GRNs.
In this presentation we provide an introduction to the basic concepts underpinning network inference tools, and discuss some unresolved challenges in the field. As a practical example, we present our preliminary results on GRN inference of Chronic Lymphocytic Leukaemia cells, aiming at understanding the crosstalk between cancer cells and immune cells from a network perspective.

**Ulysse Herbach (INRIA, Nancy)**\
*Gene expression and regulatory networks: bridging the gap between mechanistic modeling and statistical learning*\
Inferring graphs of interactions between genes has become a textbook case for high-dimensional statistics, while models describing gene expression at the molecular level have come into their own with the advent of single-cell data. Linking these two approaches seems crucial today, but the dialogue is far from obvious: statistical models often suffer from a lack of biological interpretability, and mechanistic models are known to be difficult to calibrate from real data. We recently introduced two strategies that exploit time-course scRNA-seq data, where single-cell profiling is performed after a stimulus: Harissa, a mechanistic network model driven by transcriptional bursting, and Cardamom, a scalable inference method seen as model calibration. Thanks to this correspondence, I will show that it is possible to combine the two approaches so that the same dynamical model can be used simultaneously as an inference tool, to reconstruct biologically relevant causal networks, and as a simulation tool, to generate realistic transcriptional profiles in a non-trivial way through gene interactions.

**Anaïs Baudot (MMG, Univ Aix Marseille)**\
*Network-based multimodal data integration*\
Recent technological advances and the growing availability of biomedical datasets offer unprecedented opportunities to better understand human diseases. However, translating the sheer volume and heterogeneity of these data into meaningful insights require proper computational strategies. In this presentation, I will discuss various approaches for integrating heterogeneous datasets using interaction network frameworks. Specifically, I will outline temporal, multiplex, and multilayer networks, and their associated exploration algorithms, such as community identification, random walks, or network embedding. I will illustrate the application of these different algorithms in the context of the analysis of human diseases.


**Mathieu Hemery (Inria Saclay)**\
*Artificial CRN synthesis from function specification and natural CRN repository screening*\
“What I cannot build. I do not understand.” ― Richard Feynman\
BioChemical Reactions Network (CRN) are far from random graphs. They are highly structured networks that have evolved to fulfill biological functions in order to increase the reproductive success of their organisms. Focusing on a mathematical definition of these functions, we will see how to build from scratch an abstract CRN able to implement them, be it through in silico evolution or direct compilation algorithm. Then using some graph tools may help us scan a large biomodels repository to find concretization of this abstract CRN, which may eventually give us a deep insight into the structure of real biomodels.
