---
date: 2025-05-26
title: "Journ√©es annuelles 2025 du GT Bioss"
summary: "Les **26 et 27 mai 2025** √† **Paris**"
categories: "Manifestations"
---

Lieu¬†: Institut Curie, 11 rue Pierre &amp; Marie Curie, Paris
(Amphi BDD le 26 mai et amphi Curie le 27 mai)

Les journ√©es annuelles sont un point de rassemblement important pour notre GT. Elles permettent un moment d'√©change entre les chercheuses et chercheurs, d√©butant(e)s et confirm√©(e)s, pour discuter sur nos avanc√©es scientifiques r√©centes et les d√©fis √† aborder, √† la fronti√®re en informatique, math√©matique, et biologie.
Cette ann√©e encore, nous avons d√©cid√© de regrouper les √©v√®nements du GT Bioss (une journ√©e annuelle et deux demi-journ√©es th√©matiques) en une seule p√©riode. Ainsi, les journ√©es annuelles seront suivies par deux ateliers th√©matiques, sur le m√©tabolisme et l'√©cologie marine au sein de notre communaut√©.

## Inscription

Les inscriptions sont closes. Pour rappel, le repas du lundi midi est pris en charge, sous r√©serve d'inscription. Les autres repas (lundi soir et mardi) sont √† la charge des participants.

## Programme pr√©liminaire

Le programme g√©n√©ral se d√©roulera entre le lundi 26 mai √† 10h30 et le mardi 27 mai √† 17h.
Il offrira des expos√©s invit√©s et contribu√©s, ainsi que deux ateliers d'une demi-journ√©e.

Expos√©s invit√©s confirm√©s
- [√âlisabeth Remy](https://mabios.math.cnrs.fr/perso/remy.html) (CNRS, Aix-Marseille Universit√©)
- [C√©dric Lhoussaine](https://www.cristal.univ-lille.fr/en/profil/lhoussai/) (Universit√© de Lille)

### Lundi 26 mai
*Amphith√©√¢tre H√©l√®ne Martel-Massignac (anciennement amphi BDD), 11 rue Pierre et Marie Curie, Paris* - [Carte](https://www.openstreetmap.org/way/98644347#map=19/48.844200/2.343875)

  - 10h20 - 10h30 *Welcome*
  - 10h30 - 11h15: **[√âlisabeth Remy](https://mabios.math.cnrs.fr/perso/remy.html)** (CNRS, Aix-Marseille Universit√©), *Logical modeling for biological networks: recent developments to Model Revision and Leukemia Cell Resistance*
  - 11h15 - 12h15: *Contributed talks*:
    - 11h15: **Saran Pankaew** (Institut Curie), *AstroLogics : A Comprehensive Analysis Framework of Boolean Network in model ensemble*
    - 11h30: **Gustavo Maga√±a L√≥pez** (Universit√© de Bordeaux), *Statistical Refinement of Boolean Models using Perturbation Data*
    - 11h45: **Patr√≠cia Roxo** (ENS & Aix-Marseille Universit√©), *On Model Reduction of Boolean Networks*
    - 12h00: **Rebecca Ghidini** (CNRS/ENS/Inria), *Reachability Analysis for Parametric Rule-Based Models*
  - 12h15 - 13h45: *Buffet*
  - 13h45 - 14h30: **[C√©dric Lhoussaine](https://www.cristal.univ-lille.fr/en/profil/lhoussai/)** (Universit√© de Lille), *Modeling Intestinal Glucose Absorption*
  - 14h30 - 15h15: *Contributed talks*:
    - 14h30: **Malvina Marku** (CRCT), *Time-series RNA-Seq and data-driven network inference unveil dynamic cell phenotypes in Chronic Lymphocytic Leukaemia*
    - 14h45: **Carine Legrand** (H√¥pital Saint-Louis), *Infer mutation coevolution in longitudinal pre-leukemic samples from somatic variant data*
    - 15h00: **Bastien Chassagnol** (Centre de recherche du CHU de Qu√©bec, Universit√© Laval), *Digital Twins and Reproducibility: Bridging Point-and-Click Software with Open-Source Integration*
  - 15h15 - 15h45: *Break*
  - 15h45 - 17h15: *Contributed talks*:
    - 15h45: **Denis Thieffry** (Institut Curie), *Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial*
    - 16h00: **Nathalie Verdi√®re** (Universit√© du Havre), *Identifiability in networks of nonlinear dynamical systems with linear and/or nonlinear couplings*
    - 16h15: **Karim Raqbi** (Universit√© de Lille), *Modeling Glucose Absorption in Enterocytes: Reproduction and Validation of a State-of-the-Art ODE Model*
    - 16h30: **Madeleine Eyraud** (CNRS, Universit√© de Lille), *Extraction of Interspecies Influence Graphs from Logical Learning*
    - 16h45: **Marie-Eva Fabri** (Inria Lille, Universit√© de Lille), *Abstract Simulation of Partial Reaction Networks*
    - 17h00: **Thao Dang** (Verimag Grenoble), *Temporal Data Mining for Medical Anomaly Detection*
  - 17h15 - 17h30: *Wrap-up & discussion*

### Mardi 27 mai : ateliers th√©matiques
*Amphith√©√¢tre Curie, Pavillon Curie, 11 rue Pierre et Marie Curie, Paris* - [Carte](https://www.openstreetmap.org/way/56066140)

#### üåä 9h30 √† 12h00 : Sources de donn√©es pour la mod√©lisation en √©cologie marine

**Organisation :** Maxime Folschette, Cl√©mence Frioux, √âric Pelletier

Cet atelier proposera deux expos√©s invit√©s sur la mod√©lisation en √©cologie marine, avec un focus sur les sources de donn√©es ayant permis de construire ces mod√®les.

- **Programme** :
    - 9h30 - 10h30 : **Samuel Chafforn** (CNRS, LS2N, Universit√© de Nantes)
    - 10h30 : pause
    - 11h00 - 12h00 : **Sylvie Gaudron** (Sorbonne Universit√© & LOG, Wimereux)

#### ü©∫ 14h √† 16h30 : Mod√®les m√©taboliques pour l'industrie et les applications m√©dicales
  
**Organisation :** Almut Heinken, Wolfram Liebermeister, Sabine Peres

Cet atelier r√©unira des chercheur¬∑es acad√©miques et industriels autour des applications m√©dicales et biotechnologiques des mod√®les m√©taboliques. Les participant¬∑es discuteront des approches de mod√©lisation en recherche industrielle ainsi que les passerelles entre les carri√®res acad√©miques et industrielles.

- **Intervenant‚ãÖes** :
    - **Almut Heinken** (Inserm UMRS 1256 NGERE, University of Lorraine, Nancy)
    - **Jeremy Grignard** (Servier Research & Development Institut Paris-Saclay - Data Sciences & Data Management Unit)
    - **Margit Heiske** (iMEAN)

- **Programme** :
    - 14h : Expos√©s des intervenant.es et discussions libres 
    - 15h30 : Table ronde et conclusions sur 
        - Les applications m√©dicales des mod√®les m√©taboliques
        - Quels types de mod√®les / mod√©lisation sont utilis√©s dans la recherche industrielle ? 
        - La transition du milieu acad√©mique (th√®se, postdoc) vers le monde industriel
        - ‚Ä¶


## R√©sum√©s (expos√©s du lundi) / Abstracts (Monday talks)

### [√âlisabeth Remy](https://mabios.math.cnrs.fr/perso/remy.html) (CNRS, Aix-Marseille Universit√©), *Logical modeling for biological networks: recent developments to Model Revision and Leukemia Cell Resistance*
Logical modelling is a popular mathematical framework for analyzing the behavior of regulatory and signalling networks. Despite its qualitative nature, this approach effectively reveals essential dynamical properties of complex systems. In this talk, I will present two recent developments in the context of the logical formalism: a method enabling model revision based on updating schemes, and a logical model of cell resistance to retinoic acid  in Acute Promyelocytic Leukemia.

### [C√©dric Lhoussaine](https://www.cristal.univ-lille.fr/en/profil/lhoussai/) (Universit√© de Lille), *Modeling Intestinal Glucose Absorption*
Abnormal regulation of intestinal glucose absorption (IGA) is increasingly recognized as a key
contributor to the onset and progression of Type 2 Diabetes (T2D). While this hypothesis is
clinically supported, its mechanistic validation remains limited due to gaps in metabolic
modeling and the invasive nature of current diagnostic protocols. In this talk, we present
two complementary modeling approaches aimed at characterizing and validating IGA disorders.
First, we evaluate a widely referenced system of ordinary differential equations on an original
dataset of post-prandial glycemia and insulinemia in obese diabetic patients. This model
reveals significant limitations in capturing the complexity of gastro-intestinal regulation,
underscoring the need for more specific approaches. Second, we propose a novel, minimally invasive
strategy for assessing IGA based on D-xylose dynamics. D-xylose is glucose analogue with similar
absorption properties but simpler metabolic fate. We develop a multi-compartment model that
accurately reproduces D-xylose concentration profiles across various conditions and distinguishes
the roles of gastric emptying from intestinal absorption.

### Bastien Chassagnol (Centre de recherche du CHU de Qu√©bec, Universit√© Laval), *Digital Twins and Reproducibility: Bridging Point-and-Click Software*
Introduction: Digital twins ‚Äî in-silico representations of biological systems ‚Äî are generated by integrating empirical observations with established domain expertise. These virtual Doppelganger enable researchers to interrogate underlying biological processes and predict personalised responses to therapeutic interventions. Yet, despite a proliferation of computational tools, a persistent challenge lies in replicating the results of published studies, further exacerbating public scepticism towards scientific outputs [1], [2].

Methods: One promising avenue to address this reproducibility crisis is through collaborative, community-driven initiatives, with first and foremost coding events. Precisely, the purpose of the BioModels Hackathon, supported by Virtual Patient Engine and EMBL-EBI, was to curate a compendium of models up to the standards of BioModels. Specifically, the objective was to reproduce an ODE-based model of Inflammatory Bowel Disease originally presented in [3] within 48 hours with my team of 3 early-stage researchers. Subsequently, we curated the model to meet the rigorous standards of the BioModels repository and openly disseminating our results to the wider research community.
The process, however, revealed numerous roadblocks: datasets were only available as non-machine-readable PDF screenshots; no public GitHub/GitLab code repository existed for reproducing simulations, necessitating a complete reimplementation of the model‚Äôs equations; and the bespoke and unconventional nature of the modelling scenarios in [3] rendered standard point-and-click tools such as COPASI insufficient.

Results: To overcome these limitations, we extended COPASI‚Äôs capabilities using the R programming language, enabling:
    - the implementation of advanced sensitivity analyses and statistical evaluations, not available in COPASI‚Äôs native interface,
    - the integration of ggplot2 and Reactable for the automated generation of interactive, visual and tabular reporting of main paper‚Äôs insights. 

To cap it all, we deployed a reproducible Quarto website detailing our assumptions (when specificities of the methods were not properly reported within the paper), methodological adaptations, and results: https://bastienchassagnol.github.io/COPASI_Team216_Lo2016/curation.html.

Conclusion: This small project underscores the value of pairing open-source, scriptable frameworks with conventional modelling, point-and-click software, fostering reproducibility, and transparency.

[1]	A. Desai, M. Abdelhamid, and N. R. Padalkar, ‚ÄòWhat is reproducibility in artificial intelligence and machine learning research?‚Äô, *AI Mag.*, vol. 46, no. 2, p. e70004, 2025, doi: 10.1002/aaai.70004.  
[2]	L. Udesky, ‚Äò‚ÄúPublish or perish‚Äù culture blamed for reproducibility crisis‚Äô, *Nature*, Jan. 2025, doi: 10.1038/d41586-024-04253-w.  
[3]	W.-C. Lo, V. Arsenescu, R. I. Arsenescu, and A. Friedman, ‚ÄòInflammatory Bowel Disease: How Effective Is TNF-Œ± Suppression?‚Äô, *PloS One*, vol. 11, no. 11, p. e0165782, 2016, doi: 10.1371/journal.pone.0165782.

### Thao Dang (Verimag Grenoble), *Temporal Data Mining for Medical Anomaly Detection*
This presentation discusses an approach that leverage formal methods and machine learning to detect and predict anomalies from medical time series. We use Parametric Signal Temporal Logic (PSTL) formula to express features of vital signals and rely on a real dataset to learn parameters that correspond to critical events. The PSTL framework enables a characterisation of waveform features, aiming to enhance the prediction accuracy and can be combined with traditional feature extraction in machine learning. We illustrate the approach with examples in ECG monitoring and hypotension prediction.

### Madeleine Eyraud (CNRS, Universit√© de Lille), *Extraction of Interspecies Influence Graphs from Logical Learning*
The functioning of marine ecosystems depends on interactions between species, which influence essential services such as climate regulation and water quality. Understanding these interactions in phytoplankton‚Äîat the base of trophic networks ('food chains')‚Äîis crucial in the context of global change. While the effects of the non-living environment are well quantified, previous studies have shown that interspecies interactions (such as competition and allelopathy) are also critical, yet poorly understood.

An initial machine learning study based on observational data highlights the significance of these interspecies interactions. A second objective is to characterize these interactions using an explainable machine learning method (LFIT), which generates a logical program from observational data. Since this method requires a discrete representation of the data, a species-specific discretization approach based on prior knowledge was chosen and is currently under evaluation.

The logical rules derived from the constructed program make it possible to extract knowledge, notably in the form of an influence graph between species. The construction of this graph is based on the number of rule condition matches with the dataset, as well as the probability that the rule‚Äôs conclusion occurs at the next time step. Likely interspecies influences are represented as weighted directional links in the graph. These links help prioritize the importance of species interactions in their development and contribute to their characterization.

### Marie-Eva Fabri (Inria Lille, Universit√© de Lille), *Abstract Simulation of Partial Reaction Networks*
Biological systems are often modelised by reaction networks.However, this model requires a lot of information about the systems and kinetic laws of reactions. So there is a need to define reaction networks with partial kinetic information that can specify only the activators and inhibitors of each reaction but not necessarily the precise kinetic laws. This fact hampers quantitative analysis methods based on numerical simulation. An abstract simulation of a reaction network was introduced by abstracting the real numbers in their sign [Allart et al. 2023]. This approach seems sufficiently robust to complete the partial reaction network considered. However, if we naively apply this method to a partial reaction network, the transition graph obtained is too large to provide accurate information or to accurately differentiate between activators and inhibitors. We have managed to drastically reduce the size of this transition graph by suppressing most of the unwanted transitions. To achieve this, we combine the original abstract sign structure with a new abstraction based on variations over time and we restrict the transitions to respect continuity (over this new abstract semantics).We show that this abstract simulation can be unaffected by the choice of the missing kinetic information. We prove the soundness of our abstract simulation algorithm and demonstrate its efficiency on small examples.

### Rebecca Ghidini (CNRS/ENS/Inria), *Reachability Analysis for Parametric Rule-Based Models*
Biological system modeling is an iterative process where uncertainties may arise, especially in the early stages of the modeling. Static analysis tools are needed during each stage of the modeling to help modelers detect unexpected behaviors early by automatically inferring properties about the model. However, the rule-based modeling language Kappa and its static analysis tool KaSa currently lack support for incomplete models.

In this work, we extend Kappa to support incomplete models, where some rules are considered or not considered depending on the value of some boolean parameters. We also generalize the current reachability analysis of the static analyzer KaSa to these parametric models, establishing relationships between properties and parameter values. Finally, we implement and evaluate our approach on example models.

### Carine Legrand (H√¥pital Saint-Louis), *Infer mutation coevolution in longitudinal pre-leukemic samples from somatic variant data*
Clonal heterogeneity plays a major role in tumor and in normal tissues, during progression or not to a disease, and has implications for prognostic and for personalized treatment. Bulk sequencing remains a method of choice, because coverage of somatic variants is more elevated and more uniform. Subclonal inference on longitudinal samples is currently done using mutation clustering. This clustering relies on a Dirichlet distribution prior on mutation frequency, in the frame of a Bayesian approach. Serious methods add a pre-clustering step, according to the presence or absence of mutations at the different timepoints. However, subclonal reconstruction has been hampered by the absence of any rationale on the co-evolution of mutations from one timepoint to the other. Because of this, aberrant grouping of mutations still occurs routinely. In this work, I propose an improved pre-clustering of mutations based on their co-evolution along the different timepoints, grounded in adjusted statistical testing. This improvement is integrated with presence/absence clustering, mutations grouping and plotting from existing methods. I applied this method to samples from patients affected with myelodysplastic syndromes, a disorder of the normal formation of blood cells (haematopoiesis), who transitioned to acute myeloid leukemia. I also generated simulated somatic variants data, using representative scenarii for normal and altered haematopoiesis. The results outperformed current methods and yielded more plausible, parsimonious clustering in samples from patients. A fast R implementation is proposed for reproducibility and further applications.

###  Gustavo Maga√±a L√≥pez (Universit√© de Bordeaux), *Statistical Refinement of Boolean Models using Perturbation Data*
The continuous advancement of sequencing technologies continues to push forward the resolution at which biological phenomena can be studied. The advent of multimodal omics data with single-cell resolution enhances our understanding of cellular regulatory processes. Numerous analyses can be performed on these data, one of which is building predictive mathematical models. Boolean Networks are a popular modelling framework in Systems Biology because they allow reasoning about cellular regulatory processes in mechanistic terms, without having to estimate kinetic parameters. They can encompass up to thousands of genes whilst ensuring consistency with underlying quantitative models. Boolean Networks can directly incorporate information from perturbation experiments: fixing the corresponding component(s) to active/inactive allows predicting the perturbation's effect on the system's dynamics. However, inferring Boolean Network models from coarse-grained experimental data is challenging in practice due to an untractable number of solutions perfectly matching the training data. For this reason, perturbation data are paramount because they provide information that better characterises the system. Each perturbation can be considered as implicitly providing extra constraints on the definition of admissible models. When this inference tries to simultaneously account for numerous perturbation experiments, inference times may blow up and/or memory requirements may be too high. To tackle this problem, we propose a new methodology inspired by the classical machine learning approach of splitting data into train and test sets. We employ our methodology BoNesis in order to enumerate solutions compatible with prior knowledge (i.e. a Gene Regulatory Network) and the training data (wild-type plus one or more perturbation experiments). The fitness of the resulting Boolean Networks can then be assessed using held out perturbation data. To separate Boolean Networks, we propose a set of descriptive features that constitute vectorial representations of their local functions' Boolean formulae. This is a challenging task as numerous combinations of different local functions (and variations in their structure) result in equivalent dynamic behaviour (with respect to training data). To this end, our proposed set of features measures the canalising character of Boolean functions. We show that these features enable performing binary classification of Boolean Network ensembles with regards to BNs' dynamical properties and their scores on held-out data. In conclusion, we propose a new methodology leveraging a combination of symbolic AI (logic programming) and traditional machine learning techniques. We apply machine learning algorithms on model ensembles, in order to extract features that discriminate their dynamical properties. Our objective is then to translate information learnt from perturbation data into logical constraints for Boolean Network construction.

### Malvina Marku (CRCT), *Time-series RNA-Seq and data-driven network inference unveil dynamic cell phenotypes in Chronic Lymphocytic Leukaemia*
Gene regulatory networks (GRNs) and mathematical modelling are critical for understanding the complex mechanisms that underlie cellular phenotypic changes, including the identification of key driver genes/pathways, and novel therapeutic targets. In the context of the tumour microenvironment (TME), the complex interactions between immune and cancer cells give rise to a cascade of regulatory processes at different levels, defining the cellular behaviour and response to external stimuli. In the presence of cancer cells, several immune cell populations undergo cell-state transitions toward pro-tumoral phenotypes or functional exhaustion. While the cell reprogramming and state transition of immune cells are well studied, the detailed molecular description of cancer cell behaviour in response to the interactions with immune cells remains incomplete. 

In this study, we investigate the regulatory interactions underlying Chronic Lymphocytic Leukaemia (CLL) by integrating time-series RNA sequencing with data-driven GRN inference. Using an in vitro model of the TME composed of patient-derived CLL cells and immune cells, we compared monoculture and autologous culture conditions over 5 time points. Our analysis revealed that the presence of immune cells significantly alters CLL gene expression profiles, with autologous cultures showing upregulation of immune response, cytokine signalling, and metabolic pathways. We employed data-driven inference algorithms to infer GRNs based on transcription factor (TF) activity, thus capturing temporal regulatory interactions and highlighting patient-specific regulatory mechanisms. Performing network analysis we identified distinct gene modules, revealing critical pathways influenced by immune interactions, such as cytokine signaling and metabolic reprogramming. 

Our findings emphasise the role of patient heterogeneity in shaping regulatory networks, underscoring the importance of personalised approaches in CLL research. In addition, the methodological framework applied in this study offers a workflow of integrating time-series transcriptomics with GRN inference to uncover context-specific regulatory mechanisms. Lastly, the results offer valuable insights into CLL cell behaviour and provide a foundation for developing targeted therapeutic strategies aimed at disrupting key regulatory nodes within patient-specific GRNs.

### Saran Pankaew (Institut Curie), *AstroLogics : A Comprehensive Analysis Framework of Boolean Network in model ensemble*
Boolean networks (BNs) have emerged as powerful tools for modeling cellular processes in systems biology. While multiple methods exist for synthesizing Boolean models from experimental data, there remains a critical gap in tools for systematic comparison and evaluation of each BN within possible model solutions (model ensemble). Here, we present AstroLogics, a comprehensive framework for comparing and analysing models within the model ensemble. The framework addresses two fundamental aspects of Boolean networks: dynamic properties and logical functions.

AstroLogics implements three complementary approaches for analyzing dynamic properties: attractor identification, succession diagram analysis, and stochastic simulation using MaBoSS. For logical function analysis, the framework introduces a novel method for converting Boolean equations into comparable feature matrices through Disjunctive Normal Form (DNF) transformation. This enables systematic identification of constant, varied, and marker features that distinguish between model clusters.

We demonstrate the utility of AstroLogics through analysis of multiple example BNs. Our results show that AstroLogics successfully identifies differences in the model according to different sets of attractor state. Also, the framework was able to distinguish model which harbour different transient dynamical states in models which contains same sets of attractors. Using our framework, we also identified nodes and their underlying logic features that distinguish between model clusters. Finally, the framework also proved particularly valuable for analyzing complex networks, which contains many attractors states that traditional attractor analysis is computationally challenging.

AstroLogics offers a new approach to Boolean model analysis, providing researchers with tools to evaluate model ensembles and compare their dynamic behavior and logical structure. This framework aims to improve our understanding of regulatory network models and their biological implications.

### Karim Raqbi (Universit√© de Lille), *Modeling Glucose Absorption in Enterocytes: Reproduction and Validation of a State-of-the-Art ODE Model*
The enterocyte is the main cell type of the intestinal wall. As the proximal section of the intestine was found to be involved in diabetes remission through bariatric surgery, the study of glucose absorption by these cells may be critical. Although the process of glucose absorption during fasting (low glucose in the intestine) is well-known, a different mechanism occurs during the digestion but is still poorly understood. A robust in silico model backed by experimental data could discriminate the different hypothesis. We report the reproduction, replication and reuse of a state-of-the-art model from CellML to Julia. The ODE model involving 8 species, 4 non-species variables and more than 300 parameters and 200 equations was cleaned to the necessary 119 parameters and 126 equations. We reused the algebraic equations to establish a reaction-network model and managed to reproduce the figures from the paper. With our collaboration with biologists, we project to challenge the model with in vitro data.

### Patr√≠cia Roxo (ENS & Aix-Marseille Universit√©), *On Model Reduction of Boolean Networks*
This work emphasizes the role of abstract interpretation in formalizing model reduction techniques, offering a broader framework and clearer insights into the information lost or preserved during these processes. We review existing Boolean model reduction techniques and introduce a new method that ensures reachability properties in the original model are accurately reflected in the reduced model, complementing the standard existing approach of Naldi et al.'s reduction, which does not guarantee this property.

### Denis Thieffry (Institut Curie), *Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial*
This tutorial provides stepwise instructions to install the 20 tools integrated in the CoLoMoTo software suite, in order to develop reproducible dynamical analyses of logical models of complex biological molecular networks.

The tutorial specifically focuses on the analysis of a previously published model of the regulatory network controlling mammalian cell proliferation. It includes chunks of python code to reproduce several of the results and figures published in the original article and further extend these results with the help of a selection of tools included in the CoLoMoTo suite. 

The notebook covers the visualisation of the network with the tool GINsim, an attractor analysis with bioLQM, the computation of synchronous attractors with BNS, the extraction of modules from the full model, MaBoSS simulations of the wild-type model, as well as of selected mutants, and finally the delineation of compressed probabilistic state transition graphs.

The integration of all these analyses in an executable Jupyter notebook greatly eases their reproducibility, as well as the inclusion of further extensions. This notebook can further be used as a template and enriched with other ColoMoTo tools to enable comprehensive dynamical analyses of biological network models.

### Nathalie Verdi√®re (Universit√© du Havre), *Identifiability in networks of nonlinear dynamical systems with linear and/or nonlinear couplings*
The identifiability study of dynamical systems is a property that ensures the uniqueness of parameters with respect to the model's measurement(s). Several methods exist, but for nonlinear differential equations, these methods are often limited by the size of the systems.

Some recent work on network identifiability has been published, but strong constraints on the system's linearities and coupling functions are still imposed. Unfortunately, in fields like neuroscience, such restrictions are no longer applicable due to the complex dynamics of the neurons and their interactions.

This presentation aims to present a method for studying identifiability in networks composed of linear and/or nonlinear systems with linear and/or nonlinear coupling functions. Based on the observation of certain variables of interest of some nodes, it determines which subsystems are identifiable. Additionally, the method outlines the paths and steps required to identify these subsystems. It has been automated by an algorithm, implemented in Maple and applied to an example in neuroscience, a neural network of the C. elegans worm.
