<!doctype html><html><head><title>Journées annuelles 2025 du GT Bioss</title>
<meta charset=utf-8><meta name=X-UA-Compatible content="IE=edge"><meta name=google-site-verification content><meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport><meta content="telephone=no" name=format-detection><meta name=description content><meta name=renderer content="webkit"><meta name=theme-color content="#ffffff"><meta property="og:url" content="https://gt-bioss.cnrs.fr/events/jnbioss25/"><meta property="og:site_name" content="GT Bioss"><meta property="og:title" content="Journées annuelles 2025 du GT Bioss"><meta property="og:description" content="Les 26 et 27 mai 2024 à Paris"><meta property="og:locale" content="fr_FR"><meta property="og:type" content="article"><meta property="article:section" content="events"><meta property="article:published_time" content="2025-05-26T00:00:00+00:00"><meta property="article:modified_time" content="2025-05-26T00:00:00+00:00"><link type=text/css rel=stylesheet href=/vendor/css/bootstrap.min.css><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/journal.min.ea4dd62a23031df613ee03df707ef6cbfa53acf19bdc53e6ba916edc2504def4.css integrity="sha256-6k3WKiMDHfYT7gPfcH72y/pTrPGb3FPmupFu3CUE3vQ=" media=screen><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/dark-mode.min.0286a9cefa1dd841baa4957e10f75bb71af3099ccebeadaf249daebf5842ceb4.css integrity="sha256-Aoapzvod2EG6pJV+EPdbtxrzCZzOvq2vJJ2uv1hCzrQ=" media=screen><script src=/vendor/js/loadCSS.js></script><script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script><script src=/js/toc.js></script><script>console.log("Hello from 'layouts/partials/extended_head.html'")</script></head><body><div id=app><div id=sideContainer class=side-container><a class="a-block nav-head false" href=https://gt-bioss.cnrs.fr/><div class=nav-title>GT Bioss</div><div class=nav-subtitle>Groupe de travail sur la biologie systémique symbolique</div></a><div class=nav-link-list><a class="a-block nav-link-item false" href=/about>À propos
</a><a class="a-block nav-link-item false" href=/seminaires/>Séminaires mensuels
</a><a class="a-block nav-link-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block nav-link-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block nav-link-item false" href=/inscription/>Inscription
</a><a class="a-block nav-link-item false" href=/contact.html>Contact</a></div><div class=nav-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><div id=extraContainer class=extra-container><div class=pagination><a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }"><i class="material-icons pagination-action-icon">keyboard_arrow_up
</i></a><a type=button class=pagination-action id=darkModeToggleButton><span class="material-icons pagination-action-icon" id=darkModeToggleIcon>dark_mode</span></a></div></div><div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }"><div class=drawer-content><div class=drawer-menu><a class="a-block drawer-menu-item false" href=/about>À propos
</a><a class="a-block drawer-menu-item false" href=/seminaires/>Séminaires mensuels
</a><a class="a-block drawer-menu-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block drawer-menu-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block drawer-menu-item false" href=/inscription/>Inscription
</a><a class="a-block drawer-menu-item false" href=/contact.html>Contact</a></div></div></div><transition name=fade><div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div></transition><nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container"><div id=navBackground class=nav-background></div><div class="container container-narrow nav-content"><button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>menu
</i></button>
<a id=navTitle class=navbar-brand href=https://gt-bioss.cnrs.fr/>GT Bioss
</a><button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>dark_mode</i></button></div></nav><div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }"><a href=https://gt-bioss.cnrs.fr/><div class=single-column-header-title>GT Bioss</div><div class=single-column-header-subtitle>Groupe de travail sur la biologie systémique symbolique</div></a></div><div id=content><div id=streamContainer class=stream-container><div class="post-list-container post-list-container-shadow"><div class=post><div class=post-head-wrapper-text-only><div class=post-title>Journées annuelles 2025 du GT Bioss<div class=post-subtitle>Les <strong>26 et 27 mai 2024</strong> à <strong>Paris</strong></div><div class=post-meta></div></div></div><div class=post-body-wrapper><div class=post-body v-pre><p>Lieu : Pavillon Curie, 11 rue Pierre & Marie Curie, Paris
(Amphi BDD le 26 mai et amphi Curie le 27 mai)</p><p>Les journées annuelles sont un point de rassemblement important pour notre GT. Elles permettent un moment d&rsquo;échange entre les chercheuses et chercheurs, débutant(e)s et confirmé(e)s, pour discuter sur nos avancées scientifiques récentes et les défis à aborder, à la frontière en informatique, mathématique, et biologie.
Cette année encore, nous avons décidé de regrouper les évènements du GT Bioss (une journée annuelle et deux demi-journées thématiques) en une seule période. Ainsi, les journées annuelles seront suivies par deux ateliers thématiques, sur le métabolisme et l&rsquo;écologie marine au sein de notre communauté.</p><h2 id=inscription>Inscription</h2><p>Merci d&rsquo;utiliser ce <a href=https://framaforms.org/journees-du-gt-bioss-2025-ateliers-1744015134>▶️ formulaire d&rsquo;inscription ◀️</a>. L&rsquo;inscription est gratuite mais obligatoire. Le repas du lundi midi est pris en charge, sous réserve d&rsquo;inscription. Les autres repas (lundi soir et mardi) sont à votre charge.</p><h2 id=programme-préliminaire>Programme préliminaire</h2><p>Le programme général se déroulera entre le lundi 26 mai à 10h30 et le mardi 27 mai à 17h.
Il offrira des exposés invités et contribués, ainsi que deux ateliers d&rsquo;une demi-journée.</p><p>Exposés invités confirmés</p><ul><li><a href=https://mabios.math.cnrs.fr/perso/remy.html>Élisabeth Remy</a> (CNRS, Aix-Marseille Université)</li><li><a href=https://www.cristal.univ-lille.fr/en/profil/lhoussai/>Cédric Lhoussaine</a> (Université de Lille)</li></ul><h3 id=lundi-26-mai>Lundi 26 mai</h3><p><em>Amphithéâtre Hélène Martel-Massignac (anciennement amphi BDD), Pavillon Curie, 11 rue Pierre et Marie Curie, Paris</em> - <a href=https://www.openstreetmap.org/way/56066140>Carte</a></p><ul><li>10h20 - 10h30 <em>Welcome</em></li><li>10h30 - 11h15: <strong><a href=https://mabios.math.cnrs.fr/perso/remy.html>Élisabeth Remy</a></strong> (CNRS, Aix-Marseille Université)</li><li>11h15 - 12h15: <em>Contributed talks</em>:<ul><li>11h15: <strong>Saran Pankaew</strong> (Institut Curie), <em>Ensemble de modèles, clustering de simulations (théorie)</em></li><li>11h30: <strong>Gustavo Magaña López</strong> (Université de Bordeaux), <em>Statistical Refinement of Boolean Models using Perturbation Data</em></li><li>11h45: <strong>Patrícia Roxo</strong> (CNRS/ENS/Inria), <em>On Model Reduction of Boolean Networks</em></li><li>12h00: <strong>Rebecca Ghidini</strong> (CNRS/ENS/Inria), <em>Reachability Analysis for Parametric Rule-Based Models</em></li></ul></li><li>12h15 - 13h45: <em>Buffet</em></li><li>13h45 - 14h30: <strong><a href=https://www.cristal.univ-lille.fr/en/profil/lhoussai/>Cédric Lhoussaine</a></strong> (Université de Lille)</li><li>14h30 - 15h15: <em>Contributed talks</em>:<ul><li>14h30: <strong>Bastien Chassagnol</strong> (Centre de recherche du CHU de Québec, Université Laval), <em>Towards More Realistic and Biologically Faithful Models of Cell–Cell Communication Mechanisms</em></li><li>14h45: <strong>Malvina Marku</strong> (CRCT), <em>Time-series RNA-Seq and data-driven network inference unveil dynamic cell phenotypes in Chronic Lymphocytic Leukaemia</em></li><li>15h00: <strong>Carine Legrand</strong> (Hôpital Saint-Louis), <em>Infer mutation coevolution in longitudinal pre-leukemic samples from somatic variant data</em></li></ul></li><li>15h15 - 15h45: <em>Break</em></li><li>15h45 - 17h15: <em>Contributed talks</em>:<ul><li>15h45: <strong>Nathalie Verdière</strong> (Université du Havre), <em>Identifiability in networks of nonlinear dynamical systems with linear and/or nonlinear couplings</em></li><li>16h00: <strong>Madeleine Eyraud</strong> (CNRS, Université de Lille), <em>Extraction of Interspecies Influence Graphs from Logical Learning</em></li><li>16h15: <strong>Marie-Eva Fabri</strong> (Inria Lille, Université de Lille), <em>Abstract Simulation of Partial Reaction Networks</em></li><li>16h30: <strong>Karim Raqbi</strong> (Université de Lille), <em>Modeling Glucose Absorption in Enterocytes: Reproduction and Validation of a State-of-the-Art ODE Model</em></li><li>16h45: <strong>Thao Dang</strong> (Verimag Grenoble), <em>Temporal Data Mining for Medical Anomaly Detection</em></li><li>17h00: <strong>Denis Thieffry</strong> (Institut Curie), <em>Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial</em></li></ul></li><li>17h15 - 17h30: <em>Wrap-up & discussion</em></li></ul><h3 id=mardi-27-mai--ateliers-thématiques>Mardi 27 mai : ateliers thématiques</h3><p><em>Amphithéâtre Curie, Pavillon Curie, 11 rue Pierre et Marie Curie, Paris</em> - <a href=https://www.openstreetmap.org/way/56066140>Carte</a></p><h4 id=-9h30-à-12h00--sources-de-données-pour-la-modélisation-en-écologie-marine>🌊 9h30 à 12h00 : Sources de données pour la modélisation en écologie marine</h4><p>Organisation : Maxime Folschette, Clémence Frioux, Éric Pelletier</p><p>De 9h30 à 12h00, programme à venir</p><h4 id=-14h-à-16h30--modèles-métaboliques-pour-lindustrie-et-les-applications-médicales>🩺 14h à 16h30 : Modèles métaboliques pour l&rsquo;industrie et les applications médicales</h4><p><strong>Organisation :</strong> Almut Heinken, Wolfram Liebermeister, Sabine Peres</p><p>Cet atelier réunira des chercheur·es académiques et industriels autour des applications médicales et biotechnologiques des modèles métaboliques. Les participant·es discuteront des approches de modélisation en recherche industrielle ainsi que les passerelles entre les carrières académiques et industrielles.</p><ul><li><p><strong>Intervenant.es</strong>:</p><ul><li><strong>Almut Heinken</strong> (Inserm UMRS 1256 NGERE, University of Lorraine, Nancy)</li><li><strong>Jeremy Grignard</strong> (Servier Research & Development Institut Paris-Saclay - Data Sciences & Data Management Unit)</li><li><strong>Margit Heiske</strong> (iMEAN)</li></ul></li><li><p><strong>Programme</strong> :</p><ul><li>14h : Exposés des intervenant.es et discussions libres</li><li>15h30 : Table ronde et conclusions sur<ul><li>Les applications médicales des modèles métaboliques</li><li>Quels types de modèles / modélisation sont utilisés dans la recherche industrielle ?</li><li>La transition du milieu académique (thèse, postdoc) vers le monde industriel</li><li>…</li></ul></li></ul></li></ul><h2 id=résumés-exposés-du-lundi--abstracts-monday-talks>Résumés (exposés du lundi) / Abstracts (Monday talks)</h2><h3 id=gustavo-magaña-lópez-université-de-bordeaux-statistical-refinement-of-boolean-models-using-perturbation-data>Gustavo Magaña López (Université de Bordeaux), <em>Statistical Refinement of Boolean Models using Perturbation Data</em></h3><p>The continuous advancement of sequencing technologies continues to push forward the resolution at which biological phenomena can be studied. The advent of multimodal omics data with single-cell resolution enhances our understanding of cellular regulatory processes. Numerous analyses can be performed on these data, one of which is building predictive mathematical models. Boolean Networks are a popular modelling framework in Systems Biology because they allow reasoning about cellular regulatory processes in mechanistic terms, without having to estimate kinetic parameters. They can encompass up to thousands of genes whilst ensuring consistency with underlying quantitative models. Boolean Networks can directly incorporate information from perturbation experiments: fixing the corresponding component(s) to active/inactive allows predicting the perturbation&rsquo;s effect on the system&rsquo;s dynamics. However, inferring Boolean Network models from coarse-grained experimental data is challenging in practice due to an untractable number of solutions perfectly matching the training data. For this reason, perturbation data are paramount because they provide information that better characterises the system. Each perturbation can be considered as implicitly providing extra constraints on the definition of admissible models. When this inference tries to simultaneously account for numerous perturbation experiments, inference times may blow up and/or memory requirements may be too high. To tackle this problem, we propose a new methodology inspired by the classical machine learning approach of splitting data into train and test sets. We employ our methodology BoNesis in order to enumerate solutions compatible with prior knowledge (i.e. a Gene Regulatory Network) and the training data (wild-type plus one or more perturbation experiments). The fitness of the resulting Boolean Networks can then be assessed using held out perturbation data. To separate Boolean Networks, we propose a set of descriptive features that constitute vectorial representations of their local functions&rsquo; Boolean formulae. This is a challenging task as numerous combinations of different local functions (and variations in their structure) result in equivalent dynamic behaviour (with respect to training data). To this end, our proposed set of features measures the canalising character of Boolean functions. We show that these features enable performing binary classification of Boolean Network ensembles with regards to BNs&rsquo; dynamical properties and their scores on held-out data. In conclusion, we propose a new methodology leveraging a combination of symbolic AI (logic programming) and traditional machine learning techniques. We apply machine learning algorithms on model ensembles, in order to extract features that discriminate their dynamical properties. Our objective is then to translate information learnt from perturbation data into logical constraints for Boolean Network construction.</p><h3 id=patrícia-roxo-ens-paris-on-model-reduction-of-boolean-networks>Patrícia Roxo (ENS Paris), <em>On Model Reduction of Boolean Networks</em></h3><p>This work emphasizes the role of abstract interpretation in formalizing model reduction techniques, offering a broader framework and clearer insights into the information lost or preserved during these processes. We review existing Boolean model reduction techniques and introduce a new method that ensures reachability properties in the original model are accurately reflected in the reduced model, complementing the standard existing approach of Naldi et al.&rsquo;s reduction, which does not guarantee this property.</p><h3 id=rebecca-ghidini-cnrsensinria-reachability-analysis-for-parametric-rule-based-models>Rebecca Ghidini (CNRS/ENS/Inria), <em>Reachability Analysis for Parametric Rule-Based Models</em></h3><p>Biological system modeling is an iterative process where uncertainties may arise, especially in the early stages of the modeling. Static analysis tools are needed during each stage of the modeling to help modelers detect unexpected behaviors early by automatically inferring properties about the model. However, the rule-based modeling language Kappa and its static analysis tool KaSa currently lack support for incomplete models.</p><p>In this work, we extend Kappa to support incomplete models, where some rules are considered or not considered depending on the value of some boolean parameters. We also generalize the current reachability analysis of the static analyzer KaSa to these parametric models, establishing relationships between properties and parameter values. Finally, we implement and evaluate our approach on example models.</p><h3 id=malvina-marku-crct-time-series-rna-seq-and-data-driven-network-inference-unveil-dynamic-cell-phenotypes-in-chronic-lymphocytic-leukaemia>Malvina Marku (CRCT), <em>Time-series RNA-Seq and data-driven network inference unveil dynamic cell phenotypes in Chronic Lymphocytic Leukaemia</em></h3><p>Gene regulatory networks (GRNs) and mathematical modelling are critical for understanding the complex mechanisms that underlie cellular phenotypic changes, including the identification of key driver genes/pathways, and novel therapeutic targets. In the context of the tumour microenvironment (TME), the complex interactions between immune and cancer cells give rise to a cascade of regulatory processes at different levels, defining the cellular behaviour and response to external stimuli. In the presence of cancer cells, several immune cell populations undergo cell-state transitions toward pro-tumoral phenotypes or functional exhaustion. While the cell reprogramming and state transition of immune cells are well studied, the detailed molecular description of cancer cell behaviour in response to the interactions with immune cells remains incomplete.</p><p>In this study, we investigate the regulatory interactions underlying Chronic Lymphocytic Leukaemia (CLL) by integrating time-series RNA sequencing with data-driven GRN inference. Using an in vitro model of the TME composed of patient-derived CLL cells and immune cells, we compared monoculture and autologous culture conditions over 5 time points. Our analysis revealed that the presence of immune cells significantly alters CLL gene expression profiles, with autologous cultures showing upregulation of immune response, cytokine signalling, and metabolic pathways. We employed data-driven inference algorithms to infer GRNs based on transcription factor (TF) activity, thus capturing temporal regulatory interactions and highlighting patient-specific regulatory mechanisms. Performing network analysis we identified distinct gene modules, revealing critical pathways influenced by immune interactions, such as cytokine signaling and metabolic reprogramming.</p><p>Our findings emphasise the role of patient heterogeneity in shaping regulatory networks, underscoring the importance of personalised approaches in CLL research. In addition, the methodological framework applied in this study offers a workflow of integrating time-series transcriptomics with GRN inference to uncover context-specific regulatory mechanisms. Lastly, the results offer valuable insights into CLL cell behaviour and provide a foundation for developing targeted therapeutic strategies aimed at disrupting key regulatory nodes within patient-specific GRNs.</p><h3 id=carine-legrand-hôpital-saint-louis-infer-mutation-coevolution-in-longitudinal-pre-leukemic-samples-from-somatic-variant-data>Carine Legrand (Hôpital Saint-Louis), <em>Infer mutation coevolution in longitudinal pre-leukemic samples from somatic variant data</em></h3><p>Clonal heterogeneity plays a major role in tumor and in normal tissues, during progression or not to a disease, and has implications for prognostic and for personalized treatment. Bulk sequencing remains a method of choice, because coverage of somatic variants is more elevated and more uniform. Subclonal inference on longitudinal samples is currently done using mutation clustering. This clustering relies on a Dirichlet distribution prior on mutation frequency, in the frame of a Bayesian approach. Serious methods add a pre-clustering step, according to the presence or absence of mutations at the different timepoints. However, subclonal reconstruction has been hampered by the absence of any rationale on the co-evolution of mutations from one timepoint to the other. Because of this, aberrant grouping of mutations still occurs routinely. In this work, I propose an improved pre-clustering of mutations based on their co-evolution along the different timepoints, grounded in adjusted statistical testing. This improvement is integrated with presence/absence clustering, mutations grouping and plotting from existing methods. I applied this method to samples from patients affected with myelodysplastic syndromes, a disorder of the normal formation of blood cells (haematopoiesis), who transitioned to acute myeloid leukemia. I also generated simulated somatic variants data, using representative scenarii for normal and altered haematopoiesis. The results outperformed current methods and yielded more plausible, parsimonious clustering in samples from patients. A fast R implementation is proposed for reproducibility and further applications.</p><h3 id=nathalie-verdière-université-du-havre-identifiability-in-networks-of-nonlinear-dynamical-systems-with-linear-andor-nonlinear-couplings>Nathalie Verdière (Université du Havre), <em>Identifiability in networks of nonlinear dynamical systems with linear and/or nonlinear couplings</em></h3><p>The identifiability study of dynamical systems is a property that ensures the uniqueness of parameters with respect to the model&rsquo;s measurement(s). Several methods exist, but for nonlinear differential equations, these methods are often limited by the size of the systems.</p><p>Some recent work on network identifiability has been published, but strong constraints on the system&rsquo;s linearities and coupling functions are still imposed. Unfortunately, in fields like neuroscience, such restrictions are no longer applicable due to the complex dynamics of the neurons and their interactions.</p><p>This presentation aims to present a method for studying identifiability in networks composed of linear and/or nonlinear systems with linear and/or nonlinear coupling functions. Based on the observation of certain variables of interest of some nodes, it determines which subsystems are identifiable. Additionally, the method outlines the paths and steps required to identify these subsystems. It has been automated by an algorithm, implemented in Maple and applied to an example in neuroscience, a neural network of the C. elegans worm.</p><h3 id=madeleine-eyraud-cnrs-université-de-lille-extraction-of-interspecies-influence-graphs-from-logical-learning>Madeleine Eyraud (CNRS, Université de Lille), <em>Extraction of Interspecies Influence Graphs from Logical Learning</em></h3><p>The functioning of marine ecosystems depends on interactions between species, which influence essential services such as climate regulation and water quality. Understanding these interactions in phytoplankton—at the base of trophic networks (&lsquo;food chains&rsquo;)—is crucial in the context of global change. While the effects of the non-living environment are well quantified, previous studies have shown that interspecies interactions (such as competition and allelopathy) are also critical, yet poorly understood.</p><p>An initial machine learning study based on observational data highlights the significance of these interspecies interactions. A second objective is to characterize these interactions using an explainable machine learning method (LFIT), which generates a logical program from observational data. Since this method requires a discrete representation of the data, a species-specific discretization approach based on prior knowledge was chosen and is currently under evaluation.</p><p>The logical rules derived from the constructed program make it possible to extract knowledge, notably in the form of an influence graph between species. The construction of this graph is based on the number of rule condition matches with the dataset, as well as the probability that the rule’s conclusion occurs at the next time step. Likely interspecies influences are represented as weighted directional links in the graph. These links help prioritize the importance of species interactions in their development and contribute to their characterization.</p><h3 id=marie-eva-fabri-inria-lille-université-de-lille-abstract-simulation-of-partial-reaction-networks>Marie-Eva Fabri (Inria Lille, Université de Lille), <em>Abstract Simulation of Partial Reaction Networks</em></h3><p>Biological systems are often modelised by reaction networks.However, this model requires a lot of information about the systems and kinetic laws of reactions. So there is a need to define reaction networks with partial kinetic information that can specify only the activators and inhibitors of each reaction but not necessarily the precise kinetic laws. This fact hampers quantitative analysis methods based on numerical simulation. An abstract simulation of a reaction network was introduced by abstracting the real numbers in their sign [Allart et al. 2023]. This approach seems sufficiently robust to complete the partial reaction network considered. However, if we naively apply this method to a partial reaction network, the transition graph obtained is too large to provide accurate information or to accurately differentiate between activators and inhibitors. We have managed to drastically reduce the size of this transition graph by suppressing most of the unwanted transitions. To achieve this, we combine the original abstract sign structure with a new abstraction based on variations over time and we restrict the transitions to respect continuity (over this new abstract semantics).We show that this abstract simulation can be unaffected by the choice of the missing kinetic information. We prove the soundness of our abstract simulation algorithm and demonstrate its efficiency on small examples.</p><h3 id=karim-raqbi-université-de-lille-modeling-glucose-absorption-in-enterocytes-reproduction-and-validation-of-a-state-of-the-art-ode-model>Karim Raqbi (Université de Lille), <em>Modeling Glucose Absorption in Enterocytes: Reproduction and Validation of a State-of-the-Art ODE Model</em></h3><p>The enterocyte is the main cell type of the intestinal wall. As the proximal section of the intestine was found to be involved in diabetes remission through bariatric surgery, the study of glucose absorption by these cells may be critical. Although the process of glucose absorption during fasting (low glucose in the intestine) is well-known, a different mechanism occurs during the digestion but is still poorly understood. A robust in silico model backed by experimental data could discriminate the different hypothesis. We report the reproduction, replication and reuse of a state-of-the-art model from CellML to Julia. The ODE model involving 8 species, 4 non-species variables and more than 300 parameters and 200 equations was cleaned to the necessary 119 parameters and 126 equations. We reused the algebraic equations to establish a reaction-network model and managed to reproduce the figures from the paper. With our collaboration with biologists, we project to challenge the model with in vitro data.</p><h3 id=thao-dang-verimag-grenoble-temporal-data-mining-for-medical-anomaly-detection>Thao Dang (Verimag Grenoble), <em>Temporal Data Mining for Medical Anomaly Detection</em></h3><p>This presentation discusses an approach that leverage formal methods and machine learning to detect and predict anomalies from medical time series. We use Parametric Signal Temporal Logic (PSTL) formula to express features of vital signals and rely on a real dataset to learn parameters that correspond to critical events. The PSTL framework enables a characterisation of waveform features, aiming to enhance the prediction accuracy and can be combined with traditional feature extraction in machine learning. We illustrate the approach with examples in ECG monitoring and hypotension prediction.</p><h3 id=denis-thieffry-institut-curie-reproducible-boolean-model-analyses-and-simulations-with-the-colomoto-software-suite-a-tutorial>Denis Thieffry (Institut Curie), <em>Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial</em></h3><p>This tutorial provides stepwise instructions to install the 20 tools integrated in the CoLoMoTo software suite, in order to develop reproducible dynamical analyses of logical models of complex biological molecular networks.</p><p>The tutorial specifically focuses on the analysis of a previously published model of the regulatory network controlling mammalian cell proliferation. It includes chunks of python code to reproduce several of the results and figures published in the original article and further extend these results with the help of a selection of tools included in the CoLoMoTo suite.</p><p>The notebook covers the visualisation of the network with the tool GINsim, an attractor analysis with bioLQM, the computation of synchronous attractors with BNS, the extraction of modules from the full model, MaBoSS simulations of the wild-type model, as well as of selected mutants, and finally the delineation of compressed probabilistic state transition graphs.</p><p>The integration of all these analyses in an executable Jupyter notebook greatly eases their reproducibility, as well as the inclusion of further extensions. This notebook can further be used as a template and enriched with other ColoMoTo tools to enable comprehensive dynamical analyses of biological network models.</p><hr width=100% id=EOF><p style=color:#777>Dernière modification le 26/05/2025</p></div></div><nav class=post-pagination><a class=newer-posts>Suivant<br>Pas de post plus récent.
</a><a class=older-posts href=https://gt-bioss.cnrs.fr/seminaires/2025-04/>Précédent<br>Séminaire virtuel: vendredi 4 avril 2025</a></nav><div class=post-comment-wrapper></div></div></div></div></div><div id=single-column-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Basé sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapté de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><script src=/js/journal.js></script></body></html>