<!doctype html><html><head><title>Thematic workshop on networks and biological models inference</title>
<meta charset=utf-8><meta name=X-UA-Compatible content="IE=edge"><meta name=google-site-verification content><meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport><meta content="telephone=no" name=format-detection><meta name=description content><meta name=renderer content="webkit"><meta name=theme-color content="#ffffff"><meta property="og:url" content="https://gt-bioss.cnrs.fr/events/cirm2023/"><meta property="og:site_name" content="GT Bioss"><meta property="og:title" content="Thematic workshop on networks and biological models inference"><meta property="og:description" content="Le 3 et 4 juillet 2023 √† Marseille"><meta property="og:locale" content="fr_FR"><meta property="og:type" content="article"><meta property="article:section" content="events"><meta property="article:published_time" content="2023-07-03T00:00:00+00:00"><meta property="article:modified_time" content="2023-07-03T00:00:00+00:00"><link type=text/css rel=stylesheet href=/vendor/css/bootstrap.min.css><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/journal.min.ea4dd62a23031df613ee03df707ef6cbfa53acf19bdc53e6ba916edc2504def4.css integrity="sha256-6k3WKiMDHfYT7gPfcH72y/pTrPGb3FPmupFu3CUE3vQ=" media=screen><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/dark-mode.min.0286a9cefa1dd841baa4957e10f75bb71af3099ccebeadaf249daebf5842ceb4.css integrity="sha256-Aoapzvod2EG6pJV+EPdbtxrzCZzOvq2vJJ2uv1hCzrQ=" media=screen><script src=/vendor/js/loadCSS.js></script><script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script><script src=/js/toc.js></script><script>console.log("Hello from 'layouts/partials/extended_head.html'")</script></head><body><div id=app><div id=sideContainer class=side-container><a class="a-block nav-head false" href=https://gt-bioss.cnrs.fr/><div class=nav-title>GT Bioss</div><div class=nav-subtitle>Groupe de travail sur la biologie syst√©mique symbolique</div></a><div class=nav-link-list><a class="a-block nav-link-item false" href=/about>√Ä propos
</a><a class="a-block nav-link-item false" href=/seminaires/>S√©minaires mensuels
</a><a class="a-block nav-link-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block nav-link-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block nav-link-item false" href=/inscription/>Inscription
</a><a class="a-block nav-link-item false" href=/contact.html>Contact</a></div><div class=nav-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Bas√© sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapt√© de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><div id=extraContainer class=extra-container><div class=pagination><a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }"><i class="material-icons pagination-action-icon">keyboard_arrow_up
</i></a><a type=button class=pagination-action id=darkModeToggleButton><span class="material-icons pagination-action-icon" id=darkModeToggleIcon>dark_mode</span></a></div></div><div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }"><div class=drawer-content><div class=drawer-menu><a class="a-block drawer-menu-item false" href=/about>√Ä propos
</a><a class="a-block drawer-menu-item false" href=/seminaires/>S√©minaires mensuels
</a><a class="a-block drawer-menu-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block drawer-menu-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block drawer-menu-item false" href=/inscription/>Inscription
</a><a class="a-block drawer-menu-item false" href=/contact.html>Contact</a></div></div></div><transition name=fade><div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div></transition><nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container"><div id=navBackground class=nav-background></div><div class="container container-narrow nav-content"><button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>menu
</i></button>
<a id=navTitle class=navbar-brand href=https://gt-bioss.cnrs.fr/>GT Bioss
</a><button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>dark_mode</i></button></div></nav><div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }"><a href=https://gt-bioss.cnrs.fr/><div class=single-column-header-title>GT Bioss</div><div class=single-column-header-subtitle>Groupe de travail sur la biologie syst√©mique symbolique</div></a></div><div id=content><div id=streamContainer class=stream-container><div class="post-list-container post-list-container-shadow"><div class=post><div class=post-head-wrapper-text-only><div class=post-title>Thematic workshop on networks and biological models inference<div class=post-subtitle>Le <strong>3 et 4 juillet 2023</strong> √† <strong>Marseille</strong></div><div class=post-meta></div></div></div><div class=post-body-wrapper><div class=post-body v-pre><p>The aim is to gather people working on this topic to present recent results and discuss challenges and perspectives. The workshop will take place on 3 and 4 July 2023 at CIRM in Marseille.</p><p>We are grateful to GDR <a href=https://www.gdr-im.fr/>IM</a>, <a href=https://www.gdr-bim.cnrs.fr/>BIM</a>, <a href=https://www.gdria.fr/>RADIA</a>, and RT <a href=https://mathsav.math.cnrs.fr>MathSAV</a> for their financial support.</p><h3 id=registration>Registration</h3><p>‚ö†Ô∏è Accomodation and travel is at the charge of participants.</p><p>The event is full, we cannot accept additional participants. We will select and confirm the registrations very soon.</p><h3 id=venue>Venue</h3><p>The workshop takes place at <a href=https://www.cirm-math.fr>CIRM</a>, Marseille, France.</p><p>See information to reach CIRM at <a href=https://www.cirm-math.com/getting-to-cirm.html>https://www.cirm-math.com/getting-to-cirm.html</a></p><p>We recommend booking an accomodation in Marseille center, ideally close &ldquo;Rond point du Prado&rdquo; from where depart a direct bus line to CIRM.</p><h3 id=scientific-program>Scientific Program</h3><p>üñµ <a href=https://mycore.core-cloud.net/index.php/s/j8tll2vC0jkomU9>SLIDES</a></p><p><strong>Monday, July 3rd</strong></p><p>Morning session 10:00-12:00</p><ul><li>10:00 - 10:30: Malvina Marku (INSERM, CRCT), <em>Data-driven regulatory network inference, an introduction</em></li><li>10:30 - 11:00: Paola Vera-Licona (UConn Health), <em>Model checking for inferred static networks</em></li><li>11:00 - 11:30: Ulysse Herbach (Inria, Nancy), <em>Gene expression and regulatory networks: bridging the gap between mechanistic modeling and statistical learning</em></li><li>11:30 - 12:00: Maxime Folschette (Centrale Lille Institut), <em>Learning Biological Regulatory Networks from Time Series with LFIT: Theory and Practice</em></li></ul><p>Afternoon session 14:00 - 15:30</p><ul><li>14:00 - 14:30: Mathieu Bolteau (LS2N Univ Nantes), <em>Boolean networks as a framework to model human preimplantation development</em></li><li>14:30 - 15:00: Lo√Øc Paulev√© (CNRS/LaBRI Univ Bordeaux), <em>BoNesis: a declarative framework for the synthesis of Boolean networks from rich dynamical properties</em></li><li>15:00 - 15:30: Ath√©na√Øs Vaginay (Inria Nancy)</li></ul><p>Afternoon session 16:00 - 17:30</p><ul><li>16:00 - 16:30: Laetitia Gibart (I3S, Univ C√¥te d&rsquo;Azur)</li><li>16:30 - 17:00: Franck Delaplace (Paris-Saclay University - Univ. Evry), <em>TaBooN Boolean Network Synthesis Based on Tabu Search</em></li><li>17:00 - 17:30: Discussion</li></ul><p><strong>19:30</strong> Dinner at <a href=https://goo.gl/maps/GWTkWNMZHoZ8Fkx4A>Les Arcenaulx</a>, m√©tro 1 Vieux Port (‚ö†Ô∏è public transportations stops between 9pm-10pm)</p><p><strong>Tuesday, July 4rth</strong></p><p>Morning session 09:30 - 12:00</p><ul><li>09:30 - 10:00: Ana√Øs Baudot (MMG, Univ Aix Marseille), <em>Network-based multimodal data integration</em></li><li>10:00 - 10:30: Mathieu Hemery (Inria Saclay), <em>Artificial CRN synthesis from function specification and natural CRN repository screening</em></li><li>10:30 - 11:00: Fran√ßois Fages (Inria Saclay), <em>Equation discovery and chemical reaction model learning from data time series</em></li><li>11:00 - 12:00: Wrap-up discussion, and concluding remarks</li></ul><h3 id=abstracts>Abstracts</h3><p>By order of appearance</p><p><strong>Malvina Marku (INSERM, CRCT)</strong><br><em>Data-driven regulatory network inference, an introduction</em><br>Dissecting cellular reprogramming and crosstalk through gene regulatory networks
Gene regulatory networks are highly valuable conceptual models of biological systems. The inference of GRNs has posed a significant challenge in the field of systems biology for approximately two decades. In an ideal scenario, a comprehensive collection of GRNs would exist, encompassing all cell types and transcription factors, obtained through chromatin immunoprecipitation (ChIP-seq) followed by sequencing. Establishing GRNs in engineered cells would involve quantifying their overall transcription factor binding profiles and comparing them to the reference dataset. Unfortunately, such data is currently unavailable. However, recently published methods for reconstructing GRNs using genome-wide expression data have greatly improved. Additionally, repositories of gene expression have accumulated diverse biological perturbations, which are essential for accurately reconstructing GRNs.
In this presentation we provide an introduction to the basic concepts underpinning network inference tools, and discuss some unresolved challenges in the field. As a practical example, we present our preliminary results on GRN inference of Chronic Lymphocytic Leukaemia cells, aiming at understanding the crosstalk between cancer cells and immune cells from a network perspective.</p><p><strong>Paola Vera-Licona (UConn Health)</strong><br><em>Model checking for inferred static networks</em><br>After inferring the static graph representation of a gene regulatory network, it&rsquo;s crucial to ensure its accuracy. If a mathematical model of such inferred network is available, there are methods and tools to perform model checking. However, this is not always the case when only the static network (graph representation) is available. In this presentation, we will explore some strategies and tools based on semi-dynamic simulation and linkage logic theory that can be used for model checking of static networks, along with some examples.</p><p><strong>Ulysse Herbach (INRIA, Nancy)</strong><br><em>Gene expression and regulatory networks: bridging the gap between mechanistic modeling and statistical learning</em><br>Inferring graphs of interactions between genes has become a textbook case for high-dimensional statistics, while models describing gene expression at the molecular level have come into their own with the advent of single-cell data. Linking these two approaches seems crucial today, but the dialogue is far from obvious: statistical models often suffer from a lack of biological interpretability, and mechanistic models are known to be difficult to calibrate from real data. We recently introduced two strategies that exploit time-course scRNA-seq data, where single-cell profiling is performed after a stimulus: Harissa, a mechanistic network model driven by transcriptional bursting, and Cardamom, a scalable inference method seen as model calibration. Thanks to this correspondence, I will show that it is possible to combine the two approaches so that the same dynamical model can be used simultaneously as an inference tool, to reconstruct biologically relevant causal networks, and as a simulation tool, to generate realistic transcriptional profiles in a non-trivial way through gene interactions.</p><p><strong>Mathieu Bolteau (LS2N Univ Nantes)</strong><br><em>Boolean networks as a framework to model human preimplantation development</em><br>A better understanding of human embryonic development and cell fate decision is needed to improve the success rate of assisted reproductive technologies, such as in vitro fertilization (IVF). Fortunately, with novel transcriptomics technologies, vast amounts of data can now be generated, allowing the characterization of individual human embryos at a single-cell level. However, despite the potential of IVF, the current embryo culture systems and assessment methods result in a success rate of only 25%, causing emotional, social, and medical distress for both couples and the infertility medical team. Hence, novel approaches are needed to address this issue.
One such approach is the computational modeling of the human preimplantation embryo, allowing prediction of how embryos respond to specific system perturbations, such as changes in the culture media composition. Here, we search to develop a computational model to discriminate different developmental stages during embryonic development using single-cell transcriptomics (scRNAseq) data. In this presentation, I will introduce our proposed method, which involves the selection of stage-specific pseudo-perturbations to facilitate learning Boolean network models. These models are inferred from the combination of pseudo-perturbations and prior-regulatory networks, and they are optimized to fit scRNAseq data for each developmental stage accurately. I will illustrate the method&rsquo;s application in the trophectoderm (TE) maturation. Our method allows us to identify a family of Boolean networks specific to medium and late TE developmental stages, revealing opposite regulation pathways and supporting biological hypotheses in this domain. By providing a more comprehensive understanding of human embryonic development, this research has the potential to improve the success rate of assisted reproductive technologies, such as IVF.</p><p><strong>Lo√Øc Paulev√© (CNRS/LaBRI, Bordeaux)</strong><br><em>BoNesis: a declarative framework for the synthesis of Boolean networks from rich dynamical properties</em><br><a href=https://github.com/bnediction/bonesis>BoNesis</a> is a Python library which offers a declarative framework for the synthesis of Boolean networks from advanced dynamical properties, such as reachability, bifurcation, minimal trap spaces, stable states, and mutations. It combines recent theoretical advances on Boolean networks with the Most Permissive update mode and the logic programming framework of Answer-Set Programming. Its main application domain is the inference of Boolean models from bulk and single-cell gene expression data of cellular differenciation and reprogramming processes.
In this talk, I&rsquo;ll give an overview of the features of BoNesis and illustrate its application to the inference of Boolean networks from actual RNA-Seq data.</p><p><strong>Franck Delaplace (Paris-Saclay University - Univ. Evry)</strong><br><em>TaBooN Boolean Network Synthesis Based on Tabu Search</em><br>Recent developments in Omics-technologies revolutionized the investigation of biology by producing molecular data in multiple dimensions and scale. This breakthrough in biology raises the crucial issue of their interpretation based on modeling. In this undertaking, the network provides a suitable framework for modeling the interactions between molecules. A biological network comprises nodes referring to the components such as genes or proteins, and the edges/arcs formalizing interactions between them. The evolution of the interactions is then modeled by the definition of a dynamical system. Among the different network categories, the Boolean network offers a reliable qualitative framework for modeling the biological systems. Automatically synthesizing a Boolean network from experimental data, therefore, remains a necessary but challenging issue. We present TaBooN, an original work-flow for synthesizing Boolean Networks from biological data. The methodology uses the data in the form of Boolean profiles for inferring all the potential local formula inference. The set of all compatible formulas form the model space from which the most truthful model regarding biological knowledge can be found.Then the selection of the fittest model is achieved by a Tabu-search algorithm. TaBooN is an automated method for Boolean Network inference from experimental data that helps biologists synthesize a reliable model faster and assist in evaluating and optimizing the biological networks‚Äô dynamic behavior, further modeling and predictions.</p><p><strong>Ana√Øs Baudot (MMG, Univ Aix Marseille)</strong><br><em>Network-based multimodal data integration</em><br>Recent technological advances and the growing availability of biomedical datasets offer unprecedented opportunities to better understand human diseases. However, translating the sheer volume and heterogeneity of these data into meaningful insights require proper computational strategies. In this presentation, I will discuss various approaches for integrating heterogeneous datasets using interaction network frameworks. Specifically, I will outline temporal, multiplex, and multilayer networks, and their associated exploration algorithms, such as community identification, random walks, or network embedding. I will illustrate the application of these different algorithms in the context of the analysis of human diseases.</p><p><strong>Mathieu Hemery (Inria Saclay)</strong><br><em>Artificial CRN synthesis from function specification and natural CRN repository screening</em><br>‚ÄúWhat I cannot build. I do not understand.‚Äù ‚Äï Richard Feynman<br>BioChemical Reactions Network (CRN) are far from random graphs. They are highly structured networks that have evolved to fulfill biological functions in order to increase the reproductive success of their organisms. Focusing on a mathematical definition of these functions, we will see how to build from scratch an abstract CRN able to implement them, be it through in silico evolution or direct compilation algorithm. Then using some graph tools may help us scan a large biomodels repository to find concretization of this abstract CRN, which may eventually give us a deep insight into the structure of real biomodels.</p><h3 id=organization>Organization</h3><ul><li>Malvina Marku</li><li>Laurence Calzone</li><li>C√©dric Lhoussaine</li><li>Lo√Øc Paulev√©</li><li>√âlisbeth R√©my</li></ul><hr width=100% id=EOF><p style=color:#777>Derni√®re modification le 03/07/2023</p></div></div><nav class=post-pagination><a class=newer-posts href=https://gt-bioss.cnrs.fr/seminaires/2023-09/>Suivant<br>S√©minaire virtuel: vendredi 8 septembre 2023
</a><a class=older-posts href=https://gt-bioss.cnrs.fr/seminaires/2023-06/>Pr√©c√©dent<br>S√©minaire virtuel: vendredi 2 juin 2023</a></nav><div class=post-comment-wrapper></div></div></div></div></div><div id=single-column-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Bas√© sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapt√© de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><script src=/js/journal.js></script></body></html>