<!doctype html><html><head><title>Journ√©es annuelles 2024 du GT Bioss</title>
<meta charset=utf-8><meta name=X-UA-Compatible content="IE=edge"><meta name=google-site-verification content><meta content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0" name=viewport><meta content="telephone=no" name=format-detection><meta name=description content><meta name=renderer content="webkit"><meta name=theme-color content="#ffffff"><meta property="og:url" content="https://gt-bioss.cnrs.fr/events/jnbioss24/"><meta property="og:site_name" content="GT Bioss"><meta property="og:title" content="Journ√©es annuelles 2024 du GT Bioss"><meta property="og:description" content="Du 27 au 29 mai 2024 √† Paris"><meta property="og:locale" content="fr_FR"><meta property="og:type" content="article"><meta property="article:section" content="events"><meta property="article:published_time" content="2024-05-27T00:00:00+00:00"><meta property="article:modified_time" content="2024-05-27T00:00:00+00:00"><link type=text/css rel=stylesheet href=/vendor/css/bootstrap.min.css><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/journal.min.ea4dd62a23031df613ee03df707ef6cbfa53acf19bdc53e6ba916edc2504def4.css integrity="sha256-6k3WKiMDHfYT7gPfcH72y/pTrPGb3FPmupFu3CUE3vQ=" media=screen><link rel=stylesheet href=https://gt-bioss.cnrs.fr/scss/dark-mode.min.0286a9cefa1dd841baa4957e10f75bb71af3099ccebeadaf249daebf5842ceb4.css integrity="sha256-Aoapzvod2EG6pJV+EPdbtxrzCZzOvq2vJJ2uv1hCzrQ=" media=screen><script src=/vendor/js/loadCSS.js></script><script>loadCSS("https://fonts.googleapis.com/css?family=Fira+Mono|Material+Icons")</script><script src=/js/toc.js></script><script>console.log("Hello from 'layouts/partials/extended_head.html'")</script></head><body><div id=app><div id=sideContainer class=side-container><a class="a-block nav-head false" href=https://gt-bioss.cnrs.fr/><div class=nav-title>GT Bioss</div><div class=nav-subtitle>Groupe de travail sur la biologie syst√©mique symbolique</div></a><div class=nav-link-list><a class="a-block nav-link-item false" href=/about>√Ä propos
</a><a class="a-block nav-link-item false" href=/seminaires/>S√©minaires mensuels
</a><a class="a-block nav-link-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block nav-link-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block nav-link-item false" href=/inscription/>Inscription
</a><a class="a-block nav-link-item false" href=/contact.html>Contact</a></div><div class=nav-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Bas√© sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapt√© de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><div id=extraContainer class=extra-container><div class=pagination><a id=globalBackToTop class="pagination-action animated-visibility" href=#top :class="{ invisible: scrollY == 0 }"><i class="material-icons pagination-action-icon">keyboard_arrow_up
</i></a><a type=button class=pagination-action id=darkModeToggleButton><span class="material-icons pagination-action-icon" id=darkModeToggleIcon>dark_mode</span></a></div></div><div class=single-column-drawer-container id=drawer v-bind:class="{ 'single-column-drawer-container-active': isDrawerOpen }"><div class=drawer-content><div class=drawer-menu><a class="a-block drawer-menu-item false" href=/about>√Ä propos
</a><a class="a-block drawer-menu-item false" href=/seminaires/>S√©minaires mensuels
</a><a class="a-block drawer-menu-item false" href=/categories/manifestations/>Manifestations
</a><a class="a-block drawer-menu-item false" href=https://mygdr.hosted.lip6.fr/GTView/11/>Membres
</a><a class="a-block drawer-menu-item false" href=/inscription/>Inscription
</a><a class="a-block drawer-menu-item false" href=/contact.html>Contact</a></div></div></div><transition name=fade><div id=drawer-mask v-bind:class="{ 'single-column-drawer-mask': mounted }" v-if=isDrawerOpen v-on:click=toggleDrawer></div></transition><nav id=navBar class="navbar sticky-top navbar-light single-column-nav-container"><div id=navBackground class=nav-background></div><div class="container container-narrow nav-content"><button id=nav_dropdown_btn class=nav-dropdown-toggle type=button v-on:click=toggleDrawer>
<i class=material-icons>menu
</i></button>
<a id=navTitle class=navbar-brand href=https://gt-bioss.cnrs.fr/>GT Bioss
</a><button type=button class=nav-darkmode-toggle id=darkModeToggleButton2>
<i class=material-icons id=darkModeToggleIcon2>dark_mode</i></button></div></nav><div class=single-column-header-container id=pageHead v-bind:style="{ transform: 'translateZ(0px) translateY('+.3*scrollY+'px)', opacity: 1-navOpacity }"><a href=https://gt-bioss.cnrs.fr/><div class=single-column-header-title>GT Bioss</div><div class=single-column-header-subtitle>Groupe de travail sur la biologie syst√©mique symbolique</div></a></div><div id=content><div id=streamContainer class=stream-container><div class="post-list-container post-list-container-shadow"><div class=post><div class=post-head-wrapper-text-only><div class=post-title>Journ√©es annuelles 2024 du GT Bioss<div class=post-subtitle>Du <strong>27 au 29 mai 2024</strong> √† <strong>Paris</strong></div><div class=post-meta></div></div></div><div class=post-body-wrapper><div class=post-body v-pre><p>Lieu¬†: <a href=https://sciences.sorbonne-universite.fr/sorbonne-universite-campus-pierre-et-marie-curie>Campus Pierre-et-Marie-Curie (Jussieu)</a>, Paris 5e</p><p>Les journ√©es annuelles sont un point de rassemblement important pour notre GT. Elles permettent un moment d&rsquo;√©change entre les chercheuses et chercheurs, d√©butant(e)s et confirm√©(e)s, pour discuter sur nos avanc√©es scientifiques r√©centes et les d√©fis √† aborder, √† la fronti√®re en informatique, math√©matique, et biologie.
Cette ann√©e, nous avons d√©cid√© de regrouper les √©v√®nements du GT Bioss (typiquement une journ√©e annuelle et 1 ou 2 journ√©es th√©matiques) en une seule p√©riode. Ainsi, les journ√©es annuelles seront suivies par deux ateliers th√©matiques, sur l&rsquo;√©valuation de m√©thodes d&rsquo;inf√©rence de mod√®les, et sur les outils d√©velopp√©s autour des mod√®les logiques.</p><h2 id=inscription>Inscription</h2><p>Les inscriptions sont closes.</p><h2 id=programme-pr√©liminaire>Programme pr√©liminaire</h2><p>Le programme g√©n√©ral se d√©roulera entre le lundi 27 mai √† 10h30 jusqu&rsquo;au mardi 28 mai √† 12h et offrira des expos√©s invit√©s et contribu√©s.
Deux ateliers d&rsquo;une demi-journ√©e suivront le mardi 28 apr√®s-midi et mercredi 29 mai matin.</p><p>Expos√©s invit√©s confirm√©s</p><ul><li><a href=https://www.inria.fr/fr/jakob-ruess>Jakob Ruess</a> (Inria Saclay)</li><li><a href=http://genome.jouy.inra.fr/~wliebermeis/index_en.html>Wolfram Liebermeister</a> (Inrae Jouy-en-Josas)</li></ul><h3 id=lundi-27-mai>Lundi 27 mai</h3><p><em>Salle de s√©minaire du LIP6 (salle 25-26/105), Campus UPMC Jussieu</em></p><ul><li>10h - 10h30 <em>Welcome coffee</em></li><li>10h30 - 11h: <a href=https://lbbe.univ-lyon1.fr/fr/annuaires-des-membres/peres-sabine>Sabine Peres</a> (LBBE, Univ Claude Bernard Lyon 1), <em>&ldquo;Constraint-based modeling of metabolic pathways using logic programming&rdquo;</em></li><li>11h - 11h30: <a href=http://maxime.folschette.name/>Maxime Folschette</a> (CRIStAL, Centrale Lille), <em>&ldquo;Modeling, Analysis and Parameter Inference of a Class of Hybrid Regulatory Networks&rdquo;</em></li><li>11h30 - 12h30: <em>contributed talks</em>:<ul><li>11h30: Sophia Orozco Ruiz (Institut Curie), <em>&ldquo;Logical modeling of Mesenchymal to Epithelial Transition in Cancer Progression&rdquo;</em></li><li>11h50: Gautier Stoll (Centre de Recherche des Cordeliers), <em>&ldquo;Logical modeling of CART-cells treatment in multiple myeloma&rdquo;</em></li><li>12h10: Nadine Ben Boina (I2M), <em>&ldquo;Logical Modeling of Dysferlinopathies&rdquo;</em></li></ul></li></ul><p>Buffet 12h30 - 14h</p><ul><li>14h - 15h: <a href=https://www.inria.fr/fr/jakob-ruess>Jakob Ruess</a> (Inria Saclay), <em>&ldquo;From single cells to microbial consortia and back: stochastic chemical kinetics coupled to population dynamics&rdquo;</em></li><li>15h - 16h50: <em>contributed talks</em>:<ul><li>15h00: Alexis Poindron (ENSTA Paris), <em>&ldquo;The unate compatibility problem&rdquo;</em></li><li>15h20: Stefan Haar (INRIA), <em>&ldquo;Bridging the discrete-continuous gap: Continuous Petri nets&rdquo;</em></li></ul></li><li>15h40 - 16h10: <em>Break</em><ul><li>16h10: Van-Giang Trinh (LIS, Aix-Marseille University), <em>&ldquo;Trap spaces of multi-valued networks: definition, computation, and applications&rdquo;</em></li><li>16h30: Mathieu Hemery (Inria Saclay), <em>&ldquo;Stabilizing CRN and Algebraic function - a surprising connexion&rdquo;</em></li></ul></li></ul><h3 id=mardi-28-mai-9h30---12h>Mardi 28 mai 9h30 - 12h</h3><p><em>Salle de s√©minaire du LIP6 (salle 25-26/105), Campus UPMC Jussieu</em></p><ul><li>9h30 - 10h30: <a href=http://genome.jouy.inra.fr/~wliebermeis/index_en.html>Wolfram Liebermeister</a> (Inrae Jouy-en-Josas), <em>&ldquo;Resource allocation in kinetic models of cell metabolism&rdquo;</em></li><li>10h30 - 11h: <em>Break</em></li><li>11h - 11h30: <em>contributed talks</em>:<ul><li>11h00: Fran√ßois FAGES (Inria Saclay), <em>&ldquo;Local vs global approaches to model learning: algorithms, failures and theorem&rdquo;</em></li><li>11h20: Tony Ribeiro (LS2N), <em>&ldquo;Learning Dynamics from Partial Observations - Symbolic Modeling of the Unknown&rdquo;</em></li></ul></li><li>11h30 - 12h Discussion</li></ul><h3 id=-atelier-√©valuation-de-m√©thodes-dinf√©rence-de-mod√®les---mardi-28-mai-14h-17h>üéØ Atelier &ldquo;√âvaluation de m√©thodes d&rsquo;inf√©rence de mod√®les&rdquo; - mardi 28 mai 14h-17h</h3><p><em>Salle 306, IBENS, ENS Paris (46 rue d‚ÄôUlm, 75005 Paris)</em></p><p>Objectifs:</p><ul><li>constituter un/des jeux de benchmarks pour √©valuer les m√©thodes d&rsquo;inf√©rence de mod√®les (graphes causaux, mod√®les dynamiques) d√©velopp√©es par des membres du GT<ul><li>diff√©rents pr√©-requis/hypoth√®ses des m√©thodes: observations √† l&rsquo;√©tat stationnaire ou non, type de donn√©es quantitatives (scRNA-Seq-like, ..), s√©ries temporelles</li><li>quels &ldquo;ground truth&rdquo;? (mod√®les de la litt√©rature, mod√®les de synth√®se, ..)</li><li>jeux de donn√©es avec mutants</li></ul></li><li>discuter de l&rsquo;√©valuation des mod√®les inf√©r√©s vis-√†-vis du ground truth:<ul><li>vis-√†-vis de la structure: √©quivalence/proximit√© avec le ground truth</li><li>vis-√†-vis des pr√©dictions (p. ex sur les mutants): permet de comparer des mod√®les de nature diff√©rente</li></ul></li></ul><p><a href=https://codimd.math.cnrs.fr/s/pR4cVqX7d>Programme d√©di√©</a></p><h3 id=-atelier-colomoto---outils-pour-la-mod√©lisation-logique---mercredi-29-9h30-12h30>üõ†Ô∏è Atelier &ldquo;CoLoMoTo - Outils pour la mod√©lisation logique&rdquo; - mercredi 29 9h30-12h30</h3><p><em>Salle 306, IBENS, ENS Paris (46 rue d‚ÄôUlm, 75005 Paris)</em></p><p>Objectifs:</p><ul><li>Faire le point sur les outils d√©velopp√©s dans le GT et leur mise √† disposition</li><li>D√©fis m√©thodologiques pour la mod√©lisation logique (combinaison de mod√®les, reproducibilit√© des simulations, &mldr;)</li><li>Faire le point sur l&rsquo;√©tat et le devenir de la distribution Docker CoLoMoTo</li></ul><p><a href=https://codimd.math.cnrs.fr/s/EVOoKlvx4>Programme¬†d√©di√©</a></p><h2 id=r√©sum√©s--abstracts>R√©sum√©s / Abstracts</h2><h3 id=jakob-ruess>Jakob Ruess</h3><p><em>From single cells to microbial consortia and back: stochastic chemical kinetics coupled to population dynamics</em></p><p>At the single-cell level, biochemical processes are inherently stochastic. Such processes are typically studied using models based on stochastic chemical kinetics, governed by a chemical master equation (CME). The CME describes the time evolution of the probability distribution over system states and has been a tremendously helpful tool in shedding light on the functioning of cellular processes. However, single cells are not living in isolation but are part of a growing population or community. In such contexts, stochasticity at the single-cell scale leads to population heterogeneity and cells may be subject to population processes, such as selection, that drive the population distribution away from the probability distribution of the single-cell process.
Here, I will introduce a multi-scale modeling framework that allows one to capture coupled stochastic single-cell and population processes. I will show that the expected population distribution of such multi-scale models can be calculated by solving a modified version of the CME that is of the same dimensionality as the standard CME. I will then show how such models can be used to explain experimental data on plasmid copy number fluctuations and population growth in media that selects against cells that have lost the plasmid. Finally, I will present an optogenetic recombination system that allows one to partition yeast populations into different cell types via external application of blue light to cells and show how our modeling framework can be used to predict and control emerging dynamics of the population composition in response to time-varying light stimuli.</p><h3 id=wolfram-liebermeister>Wolfram Liebermeister</h3><p><em>Resource allocation in kinetic models of cell metabolism</em></p><p>Metabolic fluxes in cells are often assumed to reflect an economical use of enzymes. To apply this principle of &ldquo;minimal enzyme cost&rdquo; to metabolic modelling and to study its biological consequences, I consider two types of computational models: models in which the metabolic fluxes are predefined and metabolite concentrations are optimised in order to minimise enzyme cost; and models in which the fluxes are also optimised. In the first case, optimal enzyme and metabolite levels can be computed by solving a convex optimization problem. The formalism highlights the close relationship between enzyme demand and reaction thermodynamics. In the second case, we obtain a nonlinear version of flux balance analysis, a method commonly used in flux prediction. The optimal fluxes can be determined by screening all elementary flux modes, a well-defined set of maximally sparse flux distributions. Altogether, we obtain a tractable method for predicting optimal fluxes, metabolite levels, and enzyme levels in models of central metabolism.</p><h3 id=sophia-orozco-ruiz>Sophia Orozco Ruiz</h3><p><em>Logical modeling of Mesenchymal to Epithelial Transition in Cancer Progression</em></p><p>The ability of carcinoma cells to move along the epithelial mesenchymal spectrum is crucial for metastasis. Even so, the reverse process of Epithelial to Mesenchymal Transition (EMT), Mesenchymal to Epithelial Transition (MET), has not been extensively explored, as the mechanisms capable of triggering it are not clearly defined. Here, we built a literature-based regulatory network that is translated into a logical model as a tool to reproduce and test aspects of EMT and MET. Through stochastic simulations, we provide insight into how external as well as autocrine signals can be interfered with to induce MET. Our model shows that the MET inducers capable of triggering the program are mostly dependent on the mesenchymal cellular context, as they do not act universally.</p><h3 id=gautier-stoll>Gautier Stoll</h3><p><em>Logical modeling of CART-cells treatment in multiple myeloma</em></p><p>CAR T cells are T lymphocytes that have been engineered with a ‚ÄúChimeric Antigen Receptor‚Äù for targeting specifically cancer cells. They are constructed from individual patient T lymphocytes and are re-injected as treatment. In multiple myeloma, they represent a new and promising therapy. An important clinical challenge is the understanding and the optimization of such a treatment. For that, a EU project have been created, CERTAINTY - ‚ÄúA cellular immunotherapy virtual twin for personalized cancer treatment‚Äù.
For that, we created a mathematical model of CAR T cell action on tumor cell. The model is based on signaling pathways inside T cell and tumor cells, with their interactions. The mathematical approach is logical. We use UPMaBoSS tool, which is suited for time-dependent interactions between heterogeneous cell types, including their respective signaling pathways.
In the context of this EU project, there is a need for integrating data from experiments, but above all from human cohort, which could be limited in size. We will show the different possible strategies. As preliminary results, we will show how we can integrate clinical data of clinical progression.</p><h3 id=nadine-ben-boina>Nadine Ben Boina</h3><p><em>Logical Modeling of Dysferlinopathies</em></p><p>Dysferlinopathies are a group of rare muscle diseases caused by mutations in the dysferlin gene (DYSF). Patients suffer from severe weakness and skeletal muscle atrophy, resulting from impairments in muscle cell membrane repair, regeneration of muscle fibers and inflammatory response. We are currently building a logical model that encapsulates dysferlin-associated cellular processes, in order to get a better understanding of the genotype-phenotype relationship in dysferlinopathies. Ultimately, we aim to use this model to identify optimal therapeutic targets to help manage the symptoms of the diseases.</p><p>In parallel, we have developed a method to identify relevant mutlivalued refinements of Boolean models. Boolean models offer a simplified approach to studying regulatory networks, but their high level of abstraction can miss specific cellular behaviors. Multivalued refinements address this limitation by allowing for more than just two states for some components. We introduce MRBM, a method that helps identify which components in a Boolean model should be represented with multiple states in oder to capture a specific cellular behavior (dynamical property).</p><h3 id=alexis-poindron>Alexis Poindron</h3><p><em>The unate compatibility problem in regulatory Boolean networks</em></p><p>A Boolean function is unate if it is monotone increasing or monotone decreasing, in each of its entries. The aim of this paper is to investigate the existence and construction of a unate function compatible with some imposed observations. The existence problem is found to be NP Complete. A practical algorithm of inference is exposed and tested in terms of accuracy and computational complexity.</p><h3 id=stefan-haar>Stefan Haar</h3><p>Bridging the discrete-continuous gap: Continuous Petri nets</p><p>Discrete models for biological networks have produced a rich theory and efficient analysis algorithms this is true for<br>Petri nets (PNs) and Boolean Networks (BNs), which also allow mutual representations by equivalent translations.
However, their semantics have been shown to miss state changes that are possible in a continuous representation of the same system;
this has led to the introduction of Most Permissive semantics for BNs. In this talk, I will advocate the use of a different model
from older literature, Continuous Petri nets (CPNs), that combines the modeling advantages of discrete PNs with a continuous relaxation
of state evolution, for which reachability and other key properties can be efficiently decided. A survey of some recent results
(ICATPN 2024) including attractor search with CPNs will be given, along with some first examples where the attractor landscape changes
drastically under the passage to CPNs.</p><h3 id=van-giang-trinh>Van-Giang Trinh</h3><p><em>Trap spaces of multi-valued networks: definition, computation, and applications</em></p><p>Boolean Networks (BNs) are simple but efficient mathematical formalism for modelling complex biological systems. However, having only two levels of activation is sometimes not enough to fully capture the dynamics of real-world biological systems. Hence, the need for Multi-Valued Networks (MVNs), a generalization of BNs. Despite the importance of MVNs for modelling biological systems, only limited progress has been made on developing theories, analysis methods, and tools that can support them. In particular, the recent use of trap spaces in BNs made a great impact on the field of systems biology, but there has been no similar concept defined and studied for MVNs to date. In this work, we generalize the concept of trap spaces in BNs to that in MVNs. We then develop the theory and the analysis methods for trap spaces in MVNs. In particular, we implement all proposed methods in a Python package called trapmvn. Not only showing the applicability of our approach via a realistic case study, we also evaluate the time efficiency of the method on a large collection of real-world models. The experimental results confirm the time efficiency, which we believe enables more accurate analysis on larger and more complex multi-valued models.</p><p>Reference: Trinh, Van-Giang, Belaid Benhamou, Thomas Henzinger, and Samuel Pastva. &ldquo;Trap spaces of multi-valued networks: definition, computation, and applications.&rdquo; Bioinformatics 39, no. Supplement_1 (2023): i513-i522.</p><h3 id=mathieu-hemery>Mathieu Hemery</h3><p><em>Stabilizing CRN and Algebraic function - a surprising connexion</em></p><p>The Turing completeness of continuous Chemical Reaction Networks (CRNs) states
that any computable real function can be computed by a continuous CRN on a
finite set of molecular species, possibly restricted to elementary reactions,
i.e.~with at most two reactants and mass action law kinetics. To investigate
pathways like MAPK, we have introduced a more stringent notion of robust online
analog computation called Absolute Functional Robustness (AFR), to refer to
CRNs that stabilize the concentration values of some output species to the
result of one function of the input species concentrations, while allowing
arbitrary perturbations for intermediate and output species throughout the
attraction basin. We prove that the set of real functions stabilized by a CRN
with mass action law kinetics is precisely the set of real algebraic functions.
Based on this result, we develop a compiler which takes as input any algebraic
function (defined by one polynomial and one point for selecting one branch of
the algebraic curve defined by the polynomial) and generates an abstract CRN to
stabilize it.</p><h3 id=fran√ßois-fages>Fran√ßois Fages</h3><p><em>Local vs global approaches to model learning: algorithms, failures and theorem</em></p><p>Learning continuous time dynamical models of interacting agents from observed time series data is a challenge even in absence of latent variables.
In this talk, I compare global approaches such as Sparse Identification of Nonlinear Dynamical systems (SINDy) or neural networks,
to local approaches such as Reactmine which tries to infer some preponderant reactions in the different observed transitions, within a search tree.
We show that the former state-of-the-art methods fail on some very simple single trace synthetic data whereas Reactmine succeeds.
We analyze these failures in terms of the high correlation between the observables along a single trace,
and give sufficient conditions for Zhao‚Äôs irrepresentable condition, which ensures the correctness of Lasso regression.
We validate the theoretical analysis by showing that dealing with multiple traces with many zeros in the initial conditions
de-correlates the observables of the first transitions and helps SINDy to infer the correct reactions, outside our learning protocol.</p><p>Joint work with mainly Aymane El Gadhari, Julien Martinelli and Jeremy Grignard.</p><h3 id=tony-ribeiro>Tony Ribeiro</h3><p><em>Learning Dynamics from Partial Observations - Symbolic Modeling of the Unknown</em></p><p>In this presentation, we explore the challenge of learning dynamics from partial observations, where the system&rsquo;s state is only partially observable and some variables may be unknown. We introduce a novel approach that utilizes symbolic modeling to approximate the minimal rules governing the system&rsquo;s behavior, even in the presence of uncertainty. Our method ensures an overapproximation of the original state transitions and derived minimal rules, providing a robust framework for understanding complex systems under incomplete information.</p><hr width=100% id=EOF><p style=color:#777>Derni√®re modification le 27/05/2024</p></div></div><nav class=post-pagination><a class=newer-posts>Suivant<br>Pas de post plus r√©cent.
</a><a class=older-posts href=https://gt-bioss.cnrs.fr/seminaires/2024-05/>Pr√©c√©dent<br>S√©minaire virtuel: vendredi 3 mai 2024</a></nav><div class=post-comment-wrapper></div></div></div></div></div><div id=single-column-footer><p><a href=https://gt-bioss.cnrs.fr/index.xml>Flux RSS</a></p>Bas√© sur le
Theme Hugo <a href=https://github.com/amazingrise/hugo-theme-diary>Diary</a> par <a href=https://amazingrise.net>Rise</a><br>Adapt√© de <a href=https://mak1t0.cc/ target=_blank rel="noreferrer noopener">Makito</a>'s <a href=https://github.com/SumiMakito/hexo-theme-journal/ target=_blank rel="noreferrer noopener">Journal.</a><br><br>&copy;
GT Bioss.</div></div><script src=/js/journal.js></script></body></html>